
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

    <title>55. Title : Design of the genomic transformation to make dnrp1 and dbem2 strains usable for SATAY &#8212; EXPERIMENTAL JOURNAL PHD</title>
    
  <!-- Loaded before other Sphinx assets -->
  <link href="../../_static/styles/theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">
<link href="../../_static/styles/pydata-sphinx-theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">

    
  <link rel="stylesheet"
    href="../../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    <link rel="stylesheet" type="text/css" href="../../_static/pygments.css" />
    <link rel="stylesheet" href="../../_static/styles/sphinx-book-theme.css?digest=5115cc725059bd94278eecd172e13a965bf8f5a9" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/tabs.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/design-style.b7bb847fb20b106c3d81b95245e65545.min.css" />
    
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf">

    <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
    <script src="../../_static/jquery.js"></script>
    <script src="../../_static/underscore.js"></script>
    <script src="../../_static/doctools.js"></script>
    <script src="../../_static/clipboard.min.js"></script>
    <script src="../../_static/copybutton.js"></script>
    <script src="../../_static/scripts/sphinx-book-theme.js?digest=9c920249402e914e316237a7dbc6769907cce411"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../../_static/togglebutton.js"></script>
    <script async="async" kind="hypothesis" src="https://hypothes.is/embed.js"></script>
    <script kind="utterances">

    var commentsRunWhenDOMLoaded = cb => {
    if (document.readyState != 'loading') {
        cb()
    } else if (document.addEventListener) {
        document.addEventListener('DOMContentLoaded', cb)
    } else {
        document.attachEvent('onreadystatechange', function() {
        if (document.readyState == 'complete') cb()
        })
    }
}

var addUtterances = () => {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src = "https://utteranc.es/client.js";
    script.async = "async";

    script.setAttribute("repo", "leilaicruz/Experimental-journal-jupyterBOOK");
    script.setAttribute("issue-term", "pathname");
    script.setAttribute("theme", "github-light");
    script.setAttribute("label", "üí¨ comment");
    script.setAttribute("crossorigin", "anonymous");

    sections = document.querySelectorAll("div.section");
    if (sections !== null) {
        section = sections[sections.length-1];
        section.appendChild(script);
    }
}
commentsRunWhenDOMLoaded(addUtterances);
</script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../../_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../../_static/sphinx-thebe.js"></script>
    <script>window.MathJax = {"TeX": {"Macros": {"N": "\\mathbb{N}", "floor": ["\\lfloor#1\\rfloor", 1], "bmat": ["\\left[\\begin{array}"], "emat": ["\\end{array}\\right]"]}}, "options": {"processHtmlClass": "tex2jax_process|mathjax_process|math|output_area"}}</script>
    <script defer="defer" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
    <link rel="shortcut icon" href="../../_static/flowers.jpg"/>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="56. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pAl1-Cdc42-sfGFP strains (III)." href="../2020-01/2020-02-04-FACs-as-microscopy-conditions-prep-III.html" />
    <link rel="prev" title="54. Title : Comparison between the fAcs settings used for Werner in 2017 with the ones I am using currently." href="2020-02-13-Comparison-of%20Werner-FACs-settings-with-mine.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="60">
<!-- Checkboxes to toggle the left sidebar -->
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label>
<!-- Checkboxes to toggle the in-page toc -->
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">
    <div class="visually-hidden">Toggle in-page Table of Contents</div>
</label>
<!-- Headers at the top -->
<div class="announcement header-item noprint"></div>
<div class="header header-item noprint"></div>

    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<!-- Sidebar -->
<div class="bd-sidebar noprint" id="site-navigation">
    <div class="bd-sidebar__content">
        <div class="bd-sidebar__top"><div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="../../index.html">
      
        <!-- `logo` is deprecated in Sphinx 4.0, so remove this when we stop supporting 3 -->
        
      
      
      <img src="../../_static/flowers.jpg" class="logo" alt="logo">
      
      
      <h1 class="site-logo" id="site-title">EXPERIMENTAL JOURNAL PHD</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="../../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../../intro.html">
                    Experimental progress ü•º ü¶†
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  General docs
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2022-01/2022-01-02_Balance_year_2021.html">
   1. Balance of the 2021
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-01/2021-01-19-Balance-2020.html">
   3. Balance of the 2020
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-07-Balance-of-2019.html">
   4. Balance of the 2019 and outlook to the 2020
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-02-Yearly-Progress-Meeting.html">
   5. Title: Yearly Progress Meeting (Starting my 3rd PhD year)
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Protocols
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../Protocols/Media.html">
   YPD media
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../Protocols/Yeast-transformation.html">
   High efficiency Yeast Transformation
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../Protocols/DAPI-staining-fixed-cells.html">
   Yeast DAPI Staining
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-10-2022
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2022-10/2022-10-11_DAPI_all_strains4paper.html">
   6. Title :  do yeast cells show multinuclei upon CDC42 depletion? (experiment)
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-09-2022
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2022-09/2022-09-29_Test_Biotek_Microscopy_Rebuttal_paper.html">
   7. Title : do yeast cells show multinuclei upon CDC42 depletion? (test)
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-10-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-10/2021-10-29-single-colony-genetic-check-for-the-bem1-mutants.html">
   8. Title : Genetic checks for a single colony from the stock of bem1:KanMx mutants
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-09-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-09/20092021-Biotek-new-Gal-concentrations-36C.html">
   9. Title : Biotek round on new gal concentrations at 36C
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-09/28092021-Biotek-reduced-volume-to-test-for-supressors.html">
   10. Title : Biotek volume reduced during incubation at 30C to check supressors
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-09/30092021-PCR-checks-to-gal-strains.html">
   11. Title : Genetic checks for the Gal strains used in the Biotek
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-08-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-08/2021-08-01-Biotek-testing-new-dbem3-pGal.html">
   12. Title: Biotek to test ylic140 dbem3dbem1pGal strain
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-08/2021-08-11-transformation_dnrp1dbem1.html">
   13. Title: Transformation/checking on ylic136 and ylic138 with bem1::KANMX
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-08/2021-08-30-Biotek-new-Gal-concentrations.html">
   14. Title: Biotek 30C with new Gal concentration below 0.1%
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-06-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-06/2021-06-03-dbem3-transformation-on-ylic139.html">
   15. Title : Contruction dbem1dbem3+pGal from ylic139 -&gt; ylic140
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-05-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-05/06052021-biotek-all-strains-for-paper-1.html">
   16. Title: Biotek - 1st experiment for paper all strains
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-04-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-04/26-04-2021-biotek_new_dbem1_strain.html">
   17. Title : ‚ÄúBiotek experiment for new dbem1 pGal-cdc42 strain‚Äù
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-03-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-03/2021-03-01-gDNA-PCR-for-dbem3_2.html">
   18. Title : Genomic extraction and PCR of clean ylic137_2
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-03/2021-03-05-transformation_dnrp1dbem1.html">
   19. Title: Transformation on nrp1::HYGRO with bem1::KANMX
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-03/2021-03-30-dbem1-transformation-on-pgal-wt.html">
   20. Title : bem1::KanMX transformation in WT+pGal-CDC42
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-02-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-02/2021-02-12-genomic-extractions-and-pcr-ylic137-ylic138.html">
   21. Title : Genomic extractions,PCR for checking and sequencing of dbem3 and dbem3dnrp1 new strains
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-01-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-01/2021-01-25-Checking-dbem3-dbem3dnrp1-new-strains.html">
   22. Title : PCR and sequencing checks for the new strains dbem3 and dbem3dnrp1
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-01/2021-01-25-Supp-controls-sfgFP-influence-to-paper.html">
   23. Title: Supp controls on sfgFP influence on the bulk growth
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-12-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-12/2020-12-04-ylic133-ylic136-transformation-with-bem3d.html">
   24. Title: ylic133 and ylic136 transformation with bem3d::NAT
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-12/2020-12-03-Sending-DNA-to-sequence-to-Oxford.html">
   25. Samples
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-11-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-11/2020-11-27-gDNA-extraction-and-PCR-with-ywt04a.html">
   26. Title : gDNA extraction from yWT04a
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-11/2020-11-17-gDNA-extraction-and-PCR-with-yll138.html">
   27. Title : gDNA extraction of yll138
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-10-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-21-Digestion-Ligation2Sequence.html">
   28. Title : Digestion and Ligation for sequencing
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-19-Digestion-Ligation-Test-New-SATAYDNA.html">
   29. Title : Digestion and Ligation Test for new SATAY DNA
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-15-More-DNA-extraction-WT-dnrp1.html">
   30. Title: More DNA extraction of dnrp1 and WT strains
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-14-bem3d-bem3dnrp1d-strain-construction.html">
   31. Title : ylic133 and ylic136 transformation with bem3d::NAT
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-08-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-08/2020-08-24-Digestion-circularization-PCR.html">
   32. Title: Towards DNA sequencing: Digestion, Circularization and PCR of the transposon-genome junctions.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-08/2020-08-18-DNA-Prep-SATAY.html">
   33. DNA extraction from SATAY samples
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-08/2020-08-10-SATAY-bem2d-nrp1d-WT.html">
   34. Title : 20200810-SATAY for
   <span class="math notranslate nohighlight">
    \(\Delta\)
   </span>
   bem2,
   <span class="math notranslate nohighlight">
    \(\Delta\)
   </span>
   nrp1 and WT
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-07-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-07/2020-07-15-Transformation-of-ylic133-with-pBK549-for-SATAY.html">
   35. Title : Transformation of ylic133 (WT ade- ura-) with pbK549 for SATAY
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-07/2020-07-10-Transformation-of-ylic135-and-ylic136-with-pbK549.html">
   36. Title : Transformation of ylic135 and ylic136 with pbK549 for further SATAY experiment
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-07/2020-07-06-Preparation-for-PCR-check-of-BEM2-bem2.html">
   37. Title : Preparation for PCR check of BEM2 presence and absence in strain ylic135
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-06-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-22-Essential-and-not-essential-differences-in-interactors.html">
   38. Title : How different are the number of interactors of essential genes compared with non essential genes.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-17-Experimental-planing.html">
   39. Date
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-04-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-30-Checking-bem2-deletion-on-ylic133_5.html">
   43. Title: Checking bem2 deletion on the transformed strain ylic133_5
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-29-how-commons-properties-are-useful-to-predict-SL.html">
   44. Title : From common properties to Synthetic Lethality prediction
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-24-plan-of-next-experiments.html">
   45. Title: 24042020-Plan for next experiments
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-23-Machine-Learning-study.html">
   46. Title: 23042020-Summary on machine learning study
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-08-summary-FACs-experiments.html">
   47. Date
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-03-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-03/2020-03-04-FACs.html">
   50. Title : 20200305-FACs experiment with strains ywkd065, ywkd024, ywkd038 and ywkd001
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-02-2020
 </span>
</p>
<ul class="current nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="2020-02-25-FACs-starting-with-30C-shaking.html">
   51. Title : 20200227-FACs experiment with strains ywkd065, ywkd024, ywkd038 and ywkd001
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2020-02-23-ylic133-5-transformations.html">
   52. Title: Transformation of ylic133_5 with the PCR products from yll137 and yll140
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2020-02-21-hands-on-data-from-facs.html">
   53. Title :What are the output channels from flow cytometry actualling telling us about the fluorescence of every cell?
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2020-02-13-Comparison-of%20Werner-FACs-settings-with-mine.html">
   54. Title : Comparison between the fAcs settings used for Werner in 2017 with the ones I am using currently.
  </a>
 </li>
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   55. Title : Design of the genomic transformation to make dnrp1 and dbem2 strains usable for SATAY
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-01-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-02-04-FACs-as-microscopy-conditions-prep-III.html">
   56. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pAl1-Cdc42-sfGFP strains (III).
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-28-FACs-as-microscopy-conditions-prep-II.html">
   57. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pGAl1-Cdc42-sfGFP strains (II).
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-20-FACs-as-microscopy-conditions-prep.html">
   58. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pAl1-Cdc42-sfGFP strains. (I)
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-07-machine-learning-proposal-projects.html">
   59. Title: Machine learning projects proposal for MsC/BsC/internships
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-14-essential-genes-insights.html">
   60. Title:14012020-Mapping essential genes  in WT and getting insights from it.
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-10-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-09-Sanity-check-ylic133%2BpBk549.html">
   61. Title: 09102019- Sanity check of transformants ylic133+pBk549, Byk832+pBk549 to know if they suitable for SATAY. :smile:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-03-pBK549-transformation-in-ylic133-clones-III.html">
   62. Title: 03102019- III pBK549 transformation on ylic133 for further sanity check in SATAY :blush: :punch:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-01-pBK549-transformation-in-ylic133-clonesII.html">
   63. Title : 01102019- II pBK549 transformation on ylic133 for further sanity check in SATAY :pensive:
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-09-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-24-ylic134-mating-yEK7a.html">
   64. Title :
   <font color="#ee4466">
    24092019-Mating of ylic134 with yEK7a :broken_heart:
   </font>
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-24-pBK549-transformation-in-ylic133-and-Byk832.html">
   65. Title : 24092019- pBK549 transformation on ylic133 for further sanity check in SATAY. :pensive:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-12-ylic133_2%263-transformation-pLL112.html">
   66. Title : 12092019-Transformation of ylic133 with the pLL112 plasmid :pensive:
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-08-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-27-Sporulation-protocol-ylic133%2ByEK7a.html">
   67. Title: 27082019-Sporulation protocol of 8 re-streakings from ylic133_5+yEk7a :pensive:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-22-Exp_Mating-liquid-ylic133_5-yEK7a.html">
   68. Title:
   <font color="#ee4466">
    22082019 Mating using liquid drops of yEK7a and ylic133_5 :broken_heart:
   </font>
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-16-some-plots-gene_int-data.html">
   69. Title : 16082019 Script for the visualization of the correlation between number of common interactions and type of interactions
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-12-Checking-clones-ura-and-ade-growth.html">
   70. Title : 12082019-Checking clones of Byk832 and ylic33_1 for SATAY :ok:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-07-Exp-crossing-plate-yEK7a-and-ylic133_1.html">
   71. Title :
   <font color="#ee4466">
    07082019-Crossing of yEK7a with ylic133_1 :broken_heart: :pensive:
   </font>
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-05_Exp-Byk832-transformation-with-pBK549.html">
   72. Title: 05082019-Byk832 transformation with pBk549 :ok: :white_check_mark:
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-07-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-31-Extra-checking-steps-SATAY-Byk832T.html">
   73. Title: 31072019 Double check Byk832T biolog.  replicates suitability for SATAY.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-24-Strategy-to-SATAY-target-strains.html">
   74. Title : 24072019-Strategies to generate the strains I want to study using SATAY :page_with_curl:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-23-SATAY-Byk832-1st-attempt.html">
   76. Title: 29072019-SATAY 1st attempt with Byk832-T :pray: :confused: :x:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-19-glycerol-stocks-ylic133-bio-replicates.html">
   77. Title : 23072019-Glycerols stocks of ylic133 biological replicates :snowflake:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-19_Sequencing-ylic133-biol-replicates.html">
   78. Title: 22072019-Sequencing of ylic133 biological replicates :ok: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-17_Exp-Genomic-prep-ylic133-clones.html">
   79. Title : 17072019-DNA extraction of ylic133: ura3- ade2- clones anad PCR :ok: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-16-Byk832-checking-clones-ade-ura.html">
   80. Title : 15072019-Check of growth in -ade and -ura of 10 clones of Byk832+pbk549 :ok:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-16_common-interactor-genes-workflow.html">
   81. 16-07-2019-Script in python to find the most connected genes to a query gene. :milky_way:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-11_Exp_URA-kickout-transformation-ylic132_1_2.html">
   84. Title : 11072019-ylic132-1 and ylic132-2  transformation with the OEP2 :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-10_Byk832-transformation-pBk549.html">
   86. Title : 10072019-Byk832 transformation using pBK549 plasmid (:ok:)
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-08-Data-ylic132_biolog-replicates-sequencing.html">
   87. Title: 26062019-Sequencing of the URA integration in ade2 locus in ylic132 :ok: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-04_Exp_pBK549_digestion_plasmid.html">
   88. Title : 03072019-Digestion pattern of pBK549 (Succesful) :ok:
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-06-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-28_Exp_URA_kickout_transf.html">
   89. Title: 26062019-ylic132  transformation with the OEP2 (unsuccesful) :x: :disappointed:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-26_Data_Sequencing_ylic132.html">
   90. Title: 26062019-Sequencing of the URA integration in ade2 locus in ylic132 :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-25_SATAY_card.html">
   91. Title: 25062019-Experimental Card about the SATAY experiment
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-20_Exp_Byk832_transformation_pbk549.html">
   92. Title: 19062019-Byk832 (ETH strain) transformation with pbK549 (very low effciency) :pensive: :confused:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-17_Exp_URA_kickout_transf.html">
   93. Title: 14062019-ylic132  transformation with the OEP2 (unsuccesful)  :x: :disappointed:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-06_Exp_URA_Transformation.html">
   94. Title: 06062019-Yeast transformation with the OEP1 :white_check_mark: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-03_Exp_new%20primers_ade2_locus.html">
   95. Title: 03062019-Design of new primers, for the right homology arm, for transformation that avoid the overlap with the RGA1 gene.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-03_Data_Sequencing_ade2_yll3a.html">
   96. Title: 03062019-Sequencing the homology arms of yll3a strain around the ade2 gene :ok:.
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-05-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-05/2019-05-20_Exp_URA_TRANSFORMATION.html">
   97. Title: URA transformation in ADE2 locus
  </a>
 </li>
</ul>

    </div>
</nav></div>
        <div class="bd-sidebar__bottom">
             <!-- To handle the deprecated key -->
            
            <div class="navbar_extra_footer">
            Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
            </div>
            
        </div>
    </div>
    <div id="rtd-footer-container"></div>
</div>


          


          
<!-- A tiny helper pixel to detect if we've scrolled -->
<div class="sbt-scroll-pixel-helper"></div>
<!-- Main content -->
<div class="col py-0 content-container">
    
    <div class="header-article row sticky-top noprint">
        



<div class="col py-1 d-flex header-article-main">
    <div class="header-article__left">
        
        <label for="__navigation"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="right"
title="Toggle navigation"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-bars"></i>
  </span>

</label>

        
    </div>
    <div class="header-article__right">
<button onclick="toggleFullScreen()"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="bottom"
title="Fullscreen mode"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>

<div class="menu-dropdown menu-dropdown-repository-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Source repositories">
      <i class="fab fa-github"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="https://github.com/leilaicruz/Experimental-journal-jupyterBOOK"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Source repository"
>
  

<span class="headerbtn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="headerbtn__text-container">repository</span>
</a>

      </li>
      
      <li>
        <a href="https://github.com/leilaicruz/Experimental-journal-jupyterBOOK/issues/new?title=Issue%20on%20page%20%2Fjournal/2020-02/2020-02-09-dnrp1-dbem2-for-satay.html&body=Your%20issue%20content%20here."
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Open an issue"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="headerbtn__text-container">open issue</span>
</a>

      </li>
      
    </ul>
  </div>
</div>

<div class="menu-dropdown menu-dropdown-download-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Download this page">
      <i class="fas fa-download"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="../../_sources/journal/2020-02/2020-02-09-dnrp1-dbem2-for-satay.md"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Download source file"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="headerbtn__text-container">.md</span>
</a>

      </li>
      
      <li>
        
<button onclick="printPdf(this)"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="left"
title="Print to PDF"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="headerbtn__text-container">.pdf</span>
</button>

      </li>
      
    </ul>
  </div>
</div>
<label for="__page-toc"
  class="headerbtn headerbtn-page-toc"
  
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-list"></i>
  </span>

</label>

    </div>
</div>

<!-- Table of contents -->
<div class="col-md-3 bd-toc show noprint">
    <div class="tocsection onthispage pt-5 pb-3">
        <i class="fas fa-list"></i> Contents
    </div>
    <nav id="bd-toc-nav" aria-label="Page">
        <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#date">
   55.1. Date
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#objective">
   55.2. Objective
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#method-1">
   55.3. Method 1
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#method-2">
   55.4. Method 2
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#procedure-to-design-the-right-primers-for-the-oep">
     55.4.1. Procedure to design the right primers for the OEP:
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#results">
   55.5. Results
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#next-round">
     55.5.1. Next round
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#conclusion">
   55.6. Conclusion
  </a>
 </li>
</ul>

    </nav>
</div>
    </div>
    <div class="article row">
        <div class="col pl-md-3 pl-lg-5 content-container">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>Title : Design of the genomic transformation to make dnrp1 and dbem2 strains usable for SATAY</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#date">
   55.1. Date
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#objective">
   55.2. Objective
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#method-1">
   55.3. Method 1
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#method-2">
   55.4. Method 2
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#procedure-to-design-the-right-primers-for-the-oep">
     55.4.1. Procedure to design the right primers for the OEP:
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#results">
   55.5. Results
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#next-round">
     55.5.1. Next round
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#conclusion">
   55.6. Conclusion
  </a>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            <main id="main-content" role="main">
                
              <div>
                
  <section class="tex2jax_ignore mathjax_ignore" id="title-design-of-the-genomic-transformation-to-make-dnrp1-and-dbem2-strains-usable-for-satay">
<h1><span class="section-number">55. </span>Title : Design of the genomic transformation to make dnrp1 and dbem2 strains usable for SATAY<a class="headerlink" href="#title-design-of-the-genomic-transformation-to-make-dnrp1-and-dbem2-strains-usable-for-satay" title="Permalink to this headline">#</a></h1>
<section id="date">
<h2><span class="section-number">55.1. </span>Date<a class="headerlink" href="#date" title="Permalink to this headline">#</a></h2>
<p>09022020</p>
</section>
<section id="objective">
<h2><span class="section-number">55.2. </span>Objective<a class="headerlink" href="#objective" title="Permalink to this headline">#</a></h2>
<ul class="simple">
<li><p>To design an strategy to make the following strains, in order to test SATAy on them :</p>
<ul>
<li><p><span class="math notranslate nohighlight">\(\Delta nrp1\)</span> <em>ura3-</em> <em>ade2-</em></p></li>
<li><p><span class="math notranslate nohighlight">\(\Delta bem2\)</span> <em>ura3-</em> <em>ade2-</em></p></li>
</ul>
</li>
</ul>
</section>
<section id="method-1">
<h2><span class="section-number">55.3. </span>Method 1<a class="headerlink" href="#method-1" title="Permalink to this headline">#</a></h2>
<ul class="simple">
<li><p>PCR the genomic prep of yll137 with primer 1 and 6 for nrp1 deletion and transfrom it to ylic133</p></li>
<li><p>PCR the genomic prep of yll140 with prmer 1 and 6 for bem2 deletion and transform it to ylic133.</p></li>
</ul>
<blockquote>
<div><p>Pros: less PCRs</p>
</div></blockquote>
<blockquote>
<div><p>Cons: I have to do two genomic preps.</p>
</div></blockquote>
<ul class="simple">
<li><p>The strategy is to delete <em>nrp1</em> using the HYGRO cassette and <em>bem2</em> in the leu2 locus as Liedewij has in her strains, in my ylic133 strain.</p></li>
</ul>
<p>The procedure using Method 1 is as follows:</p>
<ul class="simple">
<li><p>Perform a PCR using primers 1 and 6 for each gene (nrp1 and bem2) in yll137 (for nrp1) and yll140 (for bem2).</p>
<ul>
<li><p>Try to get around 3ug of DNA after PCR for each strain to ensure plenty of transformants.</p></li>
</ul>
</li>
<li><p>Transform it into ylic133 each fragment separately.</p></li>
<li><p>Check the integration by plate.</p></li>
</ul>
</section>
<section id="method-2">
<h2><span class="section-number">55.4. </span>Method 2<a class="headerlink" href="#method-2" title="Permalink to this headline">#</a></h2>
<blockquote>
<div><p>Pros: No genomic preps</p>
</div></blockquote>
<blockquote>
<div><p>Cons: I have to do 6 more PCRs!!!.</p>
</div></blockquote>
<ul class="simple">
<li><p>The procedure consists on:</p>
<ul>
<li><p>Perform an Overlap Extension (OEP) PCR in order to get a DNA fragment in high concentration to insert in ylic133 and kick out the <em>nrp1</em> and <em>bem2</em> gene by replacing the HYGRO and LEU2 markers, respectively.</p></li>
</ul>
</li>
<li><p>The sequence for designing the primers is taken from SGD, in the sequence tab of the gene of interest to delete and its chromosome.</p>
<ul>
<li><p>First download the CDS genomic sequence of the gene of interest.</p></li>
<li><p>Then ,download the sequence of the chromosome of the gene of interest.</p></li>
<li><p>Open SnapGene and import the chromosome sequence and search for the sequence of the gene of interest.</p></li>
<li><p>Add the corresponding feature that says that in that location is where the gene X is in the chromosome.</p></li>
<li><p>Note the gene start with ATG which is the start codon Methionine and ends with a stop codon ATT.</p></li>
<li><p>Start designing the primers!</p></li>
</ul>
</li>
</ul>
<section id="procedure-to-design-the-right-primers-for-the-oep">
<h3><span class="section-number">55.4.1. </span>Procedure to design the right primers for the OEP:<a class="headerlink" href="#procedure-to-design-the-right-primers-for-the-oep" title="Permalink to this headline">#</a></h3>
<p><img alt="Sketch of how to design the 6 primers per gene deletion transformation" src="../../_images/strategy-sketch-OEP-02.png" />{#fig:primer-design}</p>
<p>In general :</p>
<ul class="simple">
<li><p>Primers should end with a G or C nucleotide.</p></li>
<li><p>Avoid AAA or TTT in the primer design because this may cause failures in the  correct annealing to the template.</p></li>
<li><p>Melting temperatures should be from 55C to 60C .</p></li>
</ul>
<p>Specific design for this OEP:</p>
<ul class="simple">
<li><p>primer 1: Upstream primer pointing towards the start of the gene to knock out.</p></li>
<li><p>primer 2: Upstream reverse pointing outside the gene of interest. The upstream homology arm (left one) should around 200-250bp. This is the length from primer 1 to primer 2. This primer has to have an overlap <a class="footnote-reference brackets" href="#id2" id="id1">1</a> that is the reverse complement of the annealed region of primer 3.</p></li>
<li><p>primer 3: Upstream primer pointing towards the marker sequence (top strand/bottom strand) in a plasmid ( I am using plasmid 32 HphMX4 for HYGRO and 105 PRS405 ) with an extra overlap sequence  that is the reverse complement of the annealed part of primer 2.</p></li>
<li><p>primer 4: downstream primer towards the marker sequence (bottom strand/top strand) with an extra overlap sequence that is the reverse complement of the annealed part of primer 5.</p></li>
<li><p>primer 5: downstream primer pointing away  the end of the gene to knock out, with the extra overlap that is the reverse complement of annealed primer 4.</p></li>
<li><p>primer 6: downstream primer pointing  towards the end of the gene to knock out.</p></li>
</ul>
<p>PCR reactions:</p>
<ol class="simple">
<li><p>PCR1: primer 1 and 2 with genomic DNA of ylic133</p></li>
<li><p>PCR2: primer 3 and 4 with DNA  of plasmid 32 or 105</p></li>
<li><p>PCR3: primer 5 and 6 with genomic DNA of ylic133</p></li>
</ol>
</section>
</section>
<section id="results">
<h2><span class="section-number">55.5. </span>Results<a class="headerlink" href="#results" title="Permalink to this headline">#</a></h2>
<p>I did Method 1:</p>
<ul>
<li><p>18022020 : genomic prep of yll137 and yll140</p>
<ul>
<li><p>I used 1ml of overnight culture for the DNA extraction</p></li>
<li><p>I used the vortex at the highest speed for the bead tubes</p></li>
<li><p>nanodrop: Low DNA concentration‚Ä¶</p>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>Strain</p></th>
<th class="head"><p>DNA</p></th>
<th class="head"><p>A260</p></th>
<th class="head"><p>A280</p></th>
<th class="head"><p>260/280</p></th>
<th class="head"><p>260/230</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>yll137</p></td>
<td><p>8.3 ng/ul</p></td>
<td><p>0.166</p></td>
<td><p>0.082</p></td>
<td><p>2.02</p></td>
<td><p>5.36</p></td>
</tr>
<tr class="row-odd"><td><p>yll140</p></td>
<td><p>12.3 ng/ul</p></td>
<td><p>0.246</p></td>
<td><p>0.125</p></td>
<td><p>1.97</p></td>
<td><p>3.46</p></td>
</tr>
</tbody>
</table>
</li>
</ul>
</li>
</ul>
<p>PCR using primer 1 and 6 for nrp1 and primer 1 and 6 for bem2 in <strong>1ul</strong> of DNA template for yll137 and yll140 respectively.</p>
<ul>
<li><p>The expected length for the PCRs are :</p>
<ul class="simple">
<li><p>2426bp for the case of nrp1::HYGRO</p></li>
<li><p>2239bp for the case of bem2::LEU2</p></li>
</ul>
</li>
<li><p>In the protocol with Phusion Polymerase I should use around 1 min o elongation time, because this polymerase extends 1kb per 20secs.</p>
<ul>
<li><p>Protocol (1h-50mins):</p>
<p>5 mins in 95C,       1 min 72C 35x cycles       hold 10C</p>
</li>
</ul>
</li>
<li><p>19022020- DNA gel</p></li>
<li><p>Results , see &#64;fig:dna-gel</p></li>
</ul>
<p><img alt="DNA gel of the PCR products" src="../../_images/PCR-yll140-succesful.png" />{#fig:dna-gel width=50%}</p>
<ul class="contains-task-list simple">
<li><p>The DNA gel is showing I dont have any product for yll137 PCR :(</p></li>
<li><p>Possibles causes:</p>
<ul>
<li><p>Low template concentration (8.3ug)</p></li>
<li><p>High anneling temperature (60C )</p></li>
</ul>
</li>
<li><p>The size for the yll140 band is bellow  the expected one , after the estmation done in SnapGene, considering that the leu marker needs 200bp around it for both promoters. Maybe that is the correct size and I was overestimating the size of the promoters.</p></li>
<li class="task-list-item"><p><input class="task-list-item-checkbox" disabled="disabled" type="checkbox"> Next step is sequencing this PCR product , to know what is actually there.</p></li>
</ul>
<section id="next-round">
<h3><span class="section-number">55.5.1. </span>Next round<a class="headerlink" href="#next-round" title="Permalink to this headline">#</a></h3>
<ul>
<li><p>Repeat PCR for yll137 with <strong>10ul</strong> of template and <strong>58C</strong> for annealing temperature. See results in &#64;fig:blank-gel</p></li>
<li><p>Include a positive control (yll3a) with the same primers 1 and 6 for nrp1 locus amplification.</p></li>
<li><p>Include a PCR for yll140 , for later sequencing.</p>
<p><img alt="Result of the next round PCR ... very sad" src="../../_images/blank-gel-yll140-3a-137-downstair-PCR-58C-annealing.png" />{#fig:blank-gel width=50%}</p>
</li>
<li><p>I return my PCR samples to another PCR round using the 1st Protocol with the anneling temperature in 60C, and I added 0.5 ul of Phusion to each sample.</p>
<ul>
<li><p>By mistake I put 5ul of Phusion in yll137 PCR tube !!! (that was after 6pm!!!)
Here the results, see &#64;fig:overnight-PCR:</p>
<p><img alt="Overnight PCR results" src="../../_images/20022020-137-3a-140-weird-sizes-bands.png" />{#fig:overnight-PCR width=50%}</p>
<ul class="simple">
<li><p>yll3a bands is not correct, the expected size should be around 2700bp and is below 1500bp.</p></li>
<li><p>For yll137 the expected band should be around 2400 bp, considering the Hygro cassette of plasmid 32 HphMX4 is in Liedewij strain yll137. The actual band is close to 2500bp, so maybe it is correct.</p></li>
</ul>
</li>
<li><p>3rd run using a TouchDown PCR , see explanation <a class="reference external" href="https://international.neb.com/faqs/0001/01/01/what-is-touchdown-pcr">here</a></p>
<blockquote>
<div><p>It is a method for increasing specificity of PCR reactions. Touchdown PCR uses a cycling program where the annealing temperature is gradually reduced (e.g. 1-2¬∞C /every second cycle). The initial annealing temperature should be several degrees above the estimated Tm of the primers. The annealing temperature is then gradually decreased until it reaches the calculated annealing temperature of the primers or some degrees below. Amplification is then continued using this annealing temperature.</p>
</div></blockquote>
<ul class="simple">
<li><p>I used 10ul template of yll137 , 2ul of yll140 and 1ul of yll3a.</p></li>
<li><p>Results , see &#64;fig:touch-down-PCR ‚Ä¶</p></li>
</ul>
<p><img alt="Results from the touch down PCR" src="../../_images/20022020-touch-down-PCR-137-3a-140.png" />{#fig:touch-down-PCR width=50%}</p>
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>- at least yll140 work and give the same size as the 1st time.
- No idea why the rest did not work..
</pre></div>
</div>
</li>
<li><p>PCR done by Els (succesful :)) , from the lab, using her set of primers , oES83 and oES84.</p>
<ul class="simple">
<li><p>10ul of PCR was used for gel imaging.</p></li>
<li><p>40ul is left for clean up.</p></li>
</ul>
<p><img alt="PCR done by Els Sweep, with her set of primers, and it was succesful :) for yll137 and yll3a" src="../../_images/els-PCR-yll137-yll3a-succesful-21022020.png" /></p>
</li>
</ul>
</li>
<li><p>After clean up , the DNA concentration was:</p></li>
</ul>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>PCR</p></th>
<th class="head"><p>DNA</p></th>
<th class="head"><p>A260</p></th>
<th class="head"><p>A280</p></th>
<th class="head"><p>260/280</p></th>
<th class="head"><p>260/230</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>yll137-83/84 primers</p></td>
<td><p>73.3 ng/ul</p></td>
<td><p>1.46</p></td>
<td><p>-</p></td>
<td><p>1.83</p></td>
<td><p>2.23</p></td>
</tr>
<tr class="row-odd"><td><p>yll137-SPY primers</p></td>
<td><p>89.9 ng/ul</p></td>
<td><p>1.79</p></td>
<td><p>-</p></td>
<td><p>1.85</p></td>
<td><p>2.18</p></td>
</tr>
</tbody>
</table>
</section>
</section>
<section id="conclusion">
<h2><span class="section-number">55.6. </span>Conclusion<a class="headerlink" href="#conclusion" title="Permalink to this headline">#</a></h2>
<ul class="simple">
<li><p>Something is weird about my primers for yll137- nrp1::hygro amplification , that do not make the right product in the PCR reaction.</p></li>
</ul>
<hr class="footnotes docutils" />
<dl class="footnote brackets">
<dt class="label" id="id2"><span class="brackets"><a class="fn-backref" href="#id1">1</a></span></dt>
<dd><p>The extra sequence should be in front of 5‚Äô of the annealed part of the primer, leaving the 3‚Äô free for the DNA polymerase to work.</p>
</dd>
</dl>
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./journal/2020-02"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            </main>
            <footer class="footer-article noprint">
                
    <!-- Previous / next buttons -->
<div class='prev-next-area'>
    <a class='left-prev' id="prev-link" href="2020-02-13-Comparison-of%20Werner-FACs-settings-with-mine.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title"><span class="section-number">54. </span>Title : Comparison between the fAcs settings used for Werner in 2017 with the ones I am using currently.</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="../2020-01/2020-02-04-FACs-as-microscopy-conditions-prep-III.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title"><span class="section-number">56. </span>Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pAl1-Cdc42-sfGFP strains (III).</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            </footer>
        </div>
    </div>
    <div class="footer-content row">
        <footer class="col footer"><p>
  
    By Leila I√±igo de la Cruz<br/>
  
      &copy; Copyright Creative Commons.<br/>
</p>
        </footer>
    </div>
    
</div>


      </div>
    </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf"></script>


  </body>
</html>