
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

    <title>69. Title : 16082019 Script for the visualization of the correlation between number of common interactions and type of interactions &#8212; EXPERIMENTAL JOURNAL PHD</title>
    
  <!-- Loaded before other Sphinx assets -->
  <link href="../../_static/styles/theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">
<link href="../../_static/styles/pydata-sphinx-theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">

    
  <link rel="stylesheet"
    href="../../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    <link rel="stylesheet" type="text/css" href="../../_static/pygments.css" />
    <link rel="stylesheet" href="../../_static/styles/sphinx-book-theme.css?digest=5115cc725059bd94278eecd172e13a965bf8f5a9" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/tabs.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/design-style.b7bb847fb20b106c3d81b95245e65545.min.css" />
    
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf">

    <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
    <script src="../../_static/jquery.js"></script>
    <script src="../../_static/underscore.js"></script>
    <script src="../../_static/doctools.js"></script>
    <script src="../../_static/clipboard.min.js"></script>
    <script src="../../_static/copybutton.js"></script>
    <script src="../../_static/scripts/sphinx-book-theme.js?digest=9c920249402e914e316237a7dbc6769907cce411"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../../_static/togglebutton.js"></script>
    <script async="async" kind="hypothesis" src="https://hypothes.is/embed.js"></script>
    <script kind="utterances">

    var commentsRunWhenDOMLoaded = cb => {
    if (document.readyState != 'loading') {
        cb()
    } else if (document.addEventListener) {
        document.addEventListener('DOMContentLoaded', cb)
    } else {
        document.attachEvent('onreadystatechange', function() {
        if (document.readyState == 'complete') cb()
        })
    }
}

var addUtterances = () => {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src = "https://utteranc.es/client.js";
    script.async = "async";

    script.setAttribute("repo", "leilaicruz/Experimental-journal-jupyterBOOK");
    script.setAttribute("issue-term", "pathname");
    script.setAttribute("theme", "github-light");
    script.setAttribute("label", "üí¨ comment");
    script.setAttribute("crossorigin", "anonymous");

    sections = document.querySelectorAll("div.section");
    if (sections !== null) {
        section = sections[sections.length-1];
        section.appendChild(script);
    }
}
commentsRunWhenDOMLoaded(addUtterances);
</script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../../_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../../_static/sphinx-thebe.js"></script>
    <link rel="shortcut icon" href="../../_static/flowers.jpg"/>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="70. Title : 12082019-Checking clones of Byk832 and ylic33_1 for SATAY :ok:" href="2019-08-12-Checking-clones-ura-and-ade-growth.html" />
    <link rel="prev" title="68. Title: 22082019 Mating using liquid drops of yEK7a and ylic133_5 :broken_heart:" href="2019-08-22-Exp_Mating-liquid-ylic133_5-yEK7a.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="60">
<!-- Checkboxes to toggle the left sidebar -->
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label>
<!-- Checkboxes to toggle the in-page toc -->
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">
    <div class="visually-hidden">Toggle in-page Table of Contents</div>
</label>
<!-- Headers at the top -->
<div class="announcement header-item noprint"></div>
<div class="header header-item noprint"></div>

    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<!-- Sidebar -->
<div class="bd-sidebar noprint" id="site-navigation">
    <div class="bd-sidebar__content">
        <div class="bd-sidebar__top"><div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="../../index.html">
      
        <!-- `logo` is deprecated in Sphinx 4.0, so remove this when we stop supporting 3 -->
        
      
      
      <img src="../../_static/flowers.jpg" class="logo" alt="logo">
      
      
      <h1 class="site-logo" id="site-title">EXPERIMENTAL JOURNAL PHD</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="../../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../../intro.html">
                    Experimental progress ü•º ü¶†
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  General docs
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2022-01/2022-01-02_Balance_year_2021.html">
   1. Balance of the 2021
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-01/2021-01-19-Balance-2020.html">
   3. Balance of the 2020
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-07-Balance-of-2019.html">
   4. Balance of the 2019 and outlook to the 2020
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-02-Yearly-Progress-Meeting.html">
   5. Title: Yearly Progress Meeting (Starting my 3rd PhD year)
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Protocols
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../Protocols/Media.html">
   YPD media
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../Protocols/Yeast-transformation.html">
   High efficiency Yeast Transformation
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../Protocols/DAPI-staining-fixed-cells.html">
   Yeast DAPI Staining
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-10-2022
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2022-10/2022-10-11_DAPI_all_strains4paper.html">
   6. Title :  do yeast cells show multinuclei upon CDC42 depletion? (experiment)
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-09-2022
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2022-09/2022-09-29_Test_Biotek_Microscopy_Rebuttal_paper.html">
   7. Title : do yeast cells show multinuclei upon CDC42 depletion? (test)
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-10-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-10/2021-10-29-single-colony-genetic-check-for-the-bem1-mutants.html">
   8. Title : Genetic checks for a single colony from the stock of bem1:KanMx mutants
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-09-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-09/20092021-Biotek-new-Gal-concentrations-36C.html">
   9. Title : Biotek round on new gal concentrations at 36C
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-09/28092021-Biotek-reduced-volume-to-test-for-supressors.html">
   10. Title : Biotek volume reduced during incubation at 30C to check supressors
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-09/30092021-PCR-checks-to-gal-strains.html">
   11. Title : Genetic checks for the Gal strains used in the Biotek
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-08-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-08/2021-08-01-Biotek-testing-new-dbem3-pGal.html">
   12. Title: Biotek to test ylic140 dbem3dbem1pGal strain
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-08/2021-08-11-transformation_dnrp1dbem1.html">
   13. Title: Transformation/checking on ylic136 and ylic138 with bem1::KANMX
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-08/2021-08-30-Biotek-new-Gal-concentrations.html">
   14. Title: Biotek 30C with new Gal concentration below 0.1%
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-06-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-06/2021-06-03-dbem3-transformation-on-ylic139.html">
   15. Title : Contruction dbem1dbem3+pGal from ylic139 -&gt; ylic140
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-05-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-05/06052021-biotek-all-strains-for-paper-1.html">
   16. Title: Biotek - 1st experiment for paper all strains
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-04-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-04/26-04-2021-biotek_new_dbem1_strain.html">
   17. Title : ‚ÄúBiotek experiment for new dbem1 pGal-cdc42 strain‚Äù
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-03-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-03/2021-03-01-gDNA-PCR-for-dbem3_2.html">
   18. Title : Genomic extraction and PCR of clean ylic137_2
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-03/2021-03-05-transformation_dnrp1dbem1.html">
   19. Title: Transformation on nrp1::HYGRO with bem1::KANMX
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-03/2021-03-30-dbem1-transformation-on-pgal-wt.html">
   20. Title : bem1::KanMX transformation in WT+pGal-CDC42
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-02-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-02/2021-02-12-genomic-extractions-and-pcr-ylic137-ylic138.html">
   21. Title : Genomic extractions,PCR for checking and sequencing of dbem3 and dbem3dnrp1 new strains
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-01-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-01/2021-01-25-Checking-dbem3-dbem3dnrp1-new-strains.html">
   22. Title : PCR and sequencing checks for the new strains dbem3 and dbem3dnrp1
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-01/2021-01-25-Supp-controls-sfgFP-influence-to-paper.html">
   23. Title: Supp controls on sfgFP influence on the bulk growth
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-12-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-12/2020-12-04-ylic133-ylic136-transformation-with-bem3d.html">
   24. Title: ylic133 and ylic136 transformation with bem3d::NAT
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-12/2020-12-03-Sending-DNA-to-sequence-to-Oxford.html">
   25. Samples
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-11-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-11/2020-11-27-gDNA-extraction-and-PCR-with-ywt04a.html">
   26. Title : gDNA extraction from yWT04a
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-11/2020-11-17-gDNA-extraction-and-PCR-with-yll138.html">
   27. Title : gDNA extraction of yll138
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-10-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-21-Digestion-Ligation2Sequence.html">
   28. Title : Digestion and Ligation for sequencing
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-19-Digestion-Ligation-Test-New-SATAYDNA.html">
   29. Title : Digestion and Ligation Test for new SATAY DNA
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-15-More-DNA-extraction-WT-dnrp1.html">
   30. Title: More DNA extraction of dnrp1 and WT strains
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-14-bem3d-bem3dnrp1d-strain-construction.html">
   31. Title : ylic133 and ylic136 transformation with bem3d::NAT
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-08-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-08/2020-08-24-Digestion-circularization-PCR.html">
   32. Title: Towards DNA sequencing: Digestion, Circularization and PCR of the transposon-genome junctions.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-08/2020-08-18-DNA-Prep-SATAY.html">
   33. DNA extraction from SATAY samples
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-08/2020-08-10-SATAY-bem2d-nrp1d-WT.html">
   34. Title : 20200810-SATAY for
   <span class="math notranslate nohighlight">
    \(\Delta\)
   </span>
   bem2,
   <span class="math notranslate nohighlight">
    \(\Delta\)
   </span>
   nrp1 and WT
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-07-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-07/2020-07-15-Transformation-of-ylic133-with-pBK549-for-SATAY.html">
   35. Title : Transformation of ylic133 (WT ade- ura-) with pbK549 for SATAY
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-07/2020-07-10-Transformation-of-ylic135-and-ylic136-with-pbK549.html">
   36. Title : Transformation of ylic135 and ylic136 with pbK549 for further SATAY experiment
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-07/2020-07-06-Preparation-for-PCR-check-of-BEM2-bem2.html">
   37. Title : Preparation for PCR check of BEM2 presence and absence in strain ylic135
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-06-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-22-Essential-and-not-essential-differences-in-interactors.html">
   38. Title : How different are the number of interactors of essential genes compared with non essential genes.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-17-Experimental-planing.html">
   39. Date
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-04-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-30-Checking-bem2-deletion-on-ylic133_5.html">
   43. Title: Checking bem2 deletion on the transformed strain ylic133_5
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-29-how-commons-properties-are-useful-to-predict-SL.html">
   44. Title : From common properties to Synthetic Lethality prediction
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-24-plan-of-next-experiments.html">
   45. Title: 24042020-Plan for next experiments
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-23-Machine-Learning-study.html">
   46. Title: 23042020-Summary on machine learning study
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-08-summary-FACs-experiments.html">
   47. Date
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-03-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-03/2020-03-04-FACs.html">
   50. Title : 20200305-FACs experiment with strains ywkd065, ywkd024, ywkd038 and ywkd001
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-02-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-25-FACs-starting-with-30C-shaking.html">
   51. Title : 20200227-FACs experiment with strains ywkd065, ywkd024, ywkd038 and ywkd001
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-23-ylic133-5-transformations.html">
   52. Title: Transformation of ylic133_5 with the PCR products from yll137 and yll140
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-21-hands-on-data-from-facs.html">
   53. Title :What are the output channels from flow cytometry actualling telling us about the fluorescence of every cell?
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-13-Comparison-of%20Werner-FACs-settings-with-mine.html">
   54. Title : Comparison between the fAcs settings used for Werner in 2017 with the ones I am using currently.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-09-dnrp1-dbem2-for-satay.html">
   55. Title : Design of the genomic transformation to make dnrp1 and dbem2 strains usable for SATAY
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-01-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-02-04-FACs-as-microscopy-conditions-prep-III.html">
   56. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pAl1-Cdc42-sfGFP strains (III).
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-28-FACs-as-microscopy-conditions-prep-II.html">
   57. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pGAl1-Cdc42-sfGFP strains (II).
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-20-FACs-as-microscopy-conditions-prep.html">
   58. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pAl1-Cdc42-sfGFP strains. (I)
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-07-machine-learning-proposal-projects.html">
   59. Title: Machine learning projects proposal for MsC/BsC/internships
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-14-essential-genes-insights.html">
   60. Title:14012020-Mapping essential genes  in WT and getting insights from it.
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-10-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-09-Sanity-check-ylic133%2BpBk549.html">
   61. Title: 09102019- Sanity check of transformants ylic133+pBk549, Byk832+pBk549 to know if they suitable for SATAY. :smile:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-03-pBK549-transformation-in-ylic133-clones-III.html">
   62. Title: 03102019- III pBK549 transformation on ylic133 for further sanity check in SATAY :blush: :punch:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-01-pBK549-transformation-in-ylic133-clonesII.html">
   63. Title : 01102019- II pBK549 transformation on ylic133 for further sanity check in SATAY :pensive:
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-09-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-24-ylic134-mating-yEK7a.html">
   64. Title :
   <font color="#ee4466">
    24092019-Mating of ylic134 with yEK7a :broken_heart:
   </font>
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-24-pBK549-transformation-in-ylic133-and-Byk832.html">
   65. Title : 24092019- pBK549 transformation on ylic133 for further sanity check in SATAY. :pensive:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-12-ylic133_2%263-transformation-pLL112.html">
   66. Title : 12092019-Transformation of ylic133 with the pLL112 plasmid :pensive:
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-08-2019
 </span>
</p>
<ul class="current nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="2019-08-27-Sporulation-protocol-ylic133%2ByEK7a.html">
   67. Title: 27082019-Sporulation protocol of 8 re-streakings from ylic133_5+yEk7a :pensive:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-08-22-Exp_Mating-liquid-ylic133_5-yEK7a.html">
   68. Title:
   <font color="#ee4466">
    22082019 Mating using liquid drops of yEK7a and ylic133_5 :broken_heart:
   </font>
  </a>
 </li>
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   69. Title : 16082019 Script for the visualization of the correlation between number of common interactions and type of interactions
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-08-12-Checking-clones-ura-and-ade-growth.html">
   70. Title : 12082019-Checking clones of Byk832 and ylic33_1 for SATAY :ok:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-08-07-Exp-crossing-plate-yEK7a-and-ylic133_1.html">
   71. Title :
   <font color="#ee4466">
    07082019-Crossing of yEK7a with ylic133_1 :broken_heart: :pensive:
   </font>
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-08-05_Exp-Byk832-transformation-with-pBK549.html">
   72. Title: 05082019-Byk832 transformation with pBk549 :ok: :white_check_mark:
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-07-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-31-Extra-checking-steps-SATAY-Byk832T.html">
   73. Title: 31072019 Double check Byk832T biolog.  replicates suitability for SATAY.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-24-Strategy-to-SATAY-target-strains.html">
   74. Title : 24072019-Strategies to generate the strains I want to study using SATAY :page_with_curl:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-23-SATAY-Byk832-1st-attempt.html">
   76. Title: 29072019-SATAY 1st attempt with Byk832-T :pray: :confused: :x:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-19-glycerol-stocks-ylic133-bio-replicates.html">
   77. Title : 23072019-Glycerols stocks of ylic133 biological replicates :snowflake:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-19_Sequencing-ylic133-biol-replicates.html">
   78. Title: 22072019-Sequencing of ylic133 biological replicates :ok: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-17_Exp-Genomic-prep-ylic133-clones.html">
   79. Title : 17072019-DNA extraction of ylic133: ura3- ade2- clones anad PCR :ok: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-16-Byk832-checking-clones-ade-ura.html">
   80. Title : 15072019-Check of growth in -ade and -ura of 10 clones of Byk832+pbk549 :ok:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-16_common-interactor-genes-workflow.html">
   81. 16-07-2019-Script in python to find the most connected genes to a query gene. :milky_way:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-11_Exp_URA-kickout-transformation-ylic132_1_2.html">
   84. Title : 11072019-ylic132-1 and ylic132-2  transformation with the OEP2 :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-10_Byk832-transformation-pBk549.html">
   86. Title : 10072019-Byk832 transformation using pBK549 plasmid (:ok:)
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-08-Data-ylic132_biolog-replicates-sequencing.html">
   87. Title: 26062019-Sequencing of the URA integration in ade2 locus in ylic132 :ok: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-04_Exp_pBK549_digestion_plasmid.html">
   88. Title : 03072019-Digestion pattern of pBK549 (Succesful) :ok:
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-06-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-28_Exp_URA_kickout_transf.html">
   89. Title: 26062019-ylic132  transformation with the OEP2 (unsuccesful) :x: :disappointed:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-26_Data_Sequencing_ylic132.html">
   90. Title: 26062019-Sequencing of the URA integration in ade2 locus in ylic132 :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-25_SATAY_card.html">
   91. Title: 25062019-Experimental Card about the SATAY experiment
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-20_Exp_Byk832_transformation_pbk549.html">
   92. Title: 19062019-Byk832 (ETH strain) transformation with pbK549 (very low effciency) :pensive: :confused:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-17_Exp_URA_kickout_transf.html">
   93. Title: 14062019-ylic132  transformation with the OEP2 (unsuccesful)  :x: :disappointed:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-06_Exp_URA_Transformation.html">
   94. Title: 06062019-Yeast transformation with the OEP1 :white_check_mark: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-03_Exp_new%20primers_ade2_locus.html">
   95. Title: 03062019-Design of new primers, for the right homology arm, for transformation that avoid the overlap with the RGA1 gene.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-03_Data_Sequencing_ade2_yll3a.html">
   96. Title: 03062019-Sequencing the homology arms of yll3a strain around the ade2 gene :ok:.
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Experimental journal-05-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-05/2019-05-20_Exp_URA_TRANSFORMATION.html">
   97. Title: URA transformation in ADE2 locus
  </a>
 </li>
</ul>

    </div>
</nav></div>
        <div class="bd-sidebar__bottom">
             <!-- To handle the deprecated key -->
            
            <div class="navbar_extra_footer">
            Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
            </div>
            
        </div>
    </div>
    <div id="rtd-footer-container"></div>
</div>


          


          
<!-- A tiny helper pixel to detect if we've scrolled -->
<div class="sbt-scroll-pixel-helper"></div>
<!-- Main content -->
<div class="col py-0 content-container">
    
    <div class="header-article row sticky-top noprint">
        



<div class="col py-1 d-flex header-article-main">
    <div class="header-article__left">
        
        <label for="__navigation"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="right"
title="Toggle navigation"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-bars"></i>
  </span>

</label>

        
    </div>
    <div class="header-article__right">
<button onclick="toggleFullScreen()"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="bottom"
title="Fullscreen mode"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>

<div class="menu-dropdown menu-dropdown-repository-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Source repositories">
      <i class="fab fa-github"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="https://github.com/leilaicruz/Experimental-journal-jupyterBOOK"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Source repository"
>
  

<span class="headerbtn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="headerbtn__text-container">repository</span>
</a>

      </li>
      
      <li>
        <a href="https://github.com/leilaicruz/Experimental-journal-jupyterBOOK/issues/new?title=Issue%20on%20page%20%2Fjournal/2019-08/2019-08-16-some-plots-gene_int-data.html&body=Your%20issue%20content%20here."
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Open an issue"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="headerbtn__text-container">open issue</span>
</a>

      </li>
      
    </ul>
  </div>
</div>

<div class="menu-dropdown menu-dropdown-download-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Download this page">
      <i class="fas fa-download"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="../../_sources/journal/2019-08/2019-08-16-some-plots-gene_int-data.md"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Download source file"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="headerbtn__text-container">.md</span>
</a>

      </li>
      
      <li>
        
<button onclick="printPdf(this)"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="left"
title="Print to PDF"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="headerbtn__text-container">.pdf</span>
</button>

      </li>
      
    </ul>
  </div>
</div>

    </div>
</div>

<!-- Table of contents -->
<div class="col-md-3 bd-toc show noprint">
</div>
    </div>
    <div class="article row">
        <div class="col pl-md-3 pl-lg-5 content-container">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>Title : 16082019 Script for the visualization of the correlation between number of common interactions and type of interactions</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                    </div>
                </div>
            </div>
            <main id="main-content" role="main">
                
              <div>
                
  <section class="tex2jax_ignore mathjax_ignore" id="title-16082019-script-for-the-visualization-of-the-correlation-between-number-of-common-interactions-and-type-of-interactions">
<h1><span class="section-number">69. </span>Title : 16082019 Script for the visualization of the correlation between number of common interactions and type of interactions<a class="headerlink" href="#title-16082019-script-for-the-visualization-of-the-correlation-between-number-of-common-interactions-and-type-of-interactions" title="Permalink to this headline">#</a></h1>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="kn">import</span> <span class="nn">pandas</span> <span class="k">as</span> <span class="nn">pd</span>
<span class="kn">import</span> <span class="nn">numpy</span> <span class="k">as</span> <span class="nn">np</span>
<span class="kn">from</span> <span class="nn">collections</span> <span class="kn">import</span> <span class="n">defaultdict</span>
<span class="kn">import</span> <span class="nn">seaborn</span> <span class="k">as</span> <span class="nn">sns</span>
<span class="kn">import</span> <span class="nn">matplotlib.pyplot</span> <span class="k">as</span> <span class="nn">plt</span>
<span class="kn">from</span> <span class="nn">statistics</span> <span class="kn">import</span> <span class="n">mean</span>
<span class="o">%</span><span class="n">matplotlib</span> <span class="n">inline</span>
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="k">def</span> <span class="nf">query_interactions</span><span class="p">(</span><span class="n">query</span><span class="p">):</span>
    <span class="n">d2</span> <span class="o">=</span> <span class="n">defaultdict</span><span class="p">(</span><span class="nb">dict</span><span class="p">)</span>
    <span class="n">data</span><span class="o">=</span><span class="n">pd</span><span class="o">.</span><span class="n">read_excel</span><span class="p">(</span><span class="sa">r</span><span class="s1">&#39;C:\Users\linigodelacruz\Documents\PhD_2018\Documentation\Calculations\data_sgd\Interaction_data_sgd_downloads.xlsx&#39;</span><span class="p">,</span><span class="n">header</span><span class="o">=</span><span class="mi">17</span><span class="p">,</span><span class="n">encoding</span><span class="o">=</span><span class="s2">&quot;utf-8-sig&quot;</span><span class="p">)</span>
    <span class="n">col_label</span><span class="o">=</span><span class="n">data</span><span class="o">.</span><span class="n">columns</span><span class="o">.</span><span class="n">values</span>
    <span class="c1">#query=[&#39;CLN1&#39;]</span>
    <span class="c1"># giant for loop</span>
    <span class="n">names1</span> <span class="o">=</span> <span class="n">query</span>
    <span class="n">i</span><span class="o">=-</span><span class="mi">1</span>
    <span class="k">for</span> <span class="n">query1</span> <span class="ow">in</span> <span class="n">names1</span><span class="p">:</span>
        <span class="c1">#filtering the table just for the value of the query</span>
        <span class="n">q1</span> <span class="o">=</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Standard_Gene_Name_(Bait)&#39;</span><span class="p">]</span><span class="o">==</span><span class="n">query1</span><span class="p">]</span>
        <span class="c1">#r=get_row(query1)</span>
        <span class="c1">#df1 = pd.DataFrame(q1, columns=[&quot;excel_number&quot;, &quot;Interactors of query1&quot;])</span>
        <span class="c1">#gb1 = df1.groupby(&quot;Interactors of query1&quot;, sort=False) # the grouping reorganizes the set alphatically, the sort by default is true!!!!!!</span>
        <span class="c1">#gb1_row = df1.groupby(&quot;excel_number&quot;,sort=False)</span>
        <span class="n">q1_interact</span><span class="o">=</span><span class="n">q1</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">3</span><span class="p">]]</span><span class="o">.</span><span class="n">unique</span><span class="p">()</span>
       <span class="c1"># a for loop for all the interactors of query</span>
        <span class="k">for</span> <span class="n">query2</span> <span class="ow">in</span> <span class="n">q1_interact</span><span class="p">:</span>

            <span class="c1">#i+=1</span>
            <span class="c1"># write in a matrix , where the 1st column are the names in q1 and the first row = q1[0]+get_query(q1[0])</span>
            <span class="n">q2</span><span class="o">=</span><span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Standard_Gene_Name_(Bait)&#39;</span><span class="p">]</span><span class="o">==</span><span class="n">query2</span><span class="p">]</span> <span class="c1">#these are get_query(q1[i])</span>

            <span class="n">q2_interact</span><span class="o">=</span><span class="n">q2</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">3</span><span class="p">]]</span><span class="o">.</span><span class="n">unique</span><span class="p">()</span>
            <span class="c1"># how to avoid that df2 overlaps with the next value?</span>
            <span class="c1">#df2 = pd.DataFrame(q2, columns=[&quot;excel raw number of q2&quot;, &quot;Interactors of query2&quot;])</span>
            <span class="c1">#gb2 = df2.groupby(&quot;Interactors of query2&quot;,sort=False)</span>

            <span class="c1">#names3 = [name3 for name3, row2 in gb2] # to visualize what is in the interactors for query 2</span>


            <span class="n">d</span> <span class="o">=</span> <span class="n">defaultdict</span><span class="p">(</span><span class="nb">int</span><span class="p">)</span>
            <span class="n">common</span> <span class="o">=</span> <span class="p">[]</span>

            <span class="k">for</span> <span class="n">name1</span>  <span class="ow">in</span> <span class="n">q2_interact</span><span class="p">:</span>
                <span class="k">if</span> <span class="n">name1</span> <span class="ow">in</span> <span class="n">q1_interact</span><span class="p">:</span> <span class="c1"># if a gene interactor of the query1 is in interactors of query 2</span>
                    <span class="n">common</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">name1</span><span class="p">)</span>
                    <span class="n">d</span><span class="p">[</span><span class="n">name1</span><span class="p">]</span> <span class="o">+=</span> <span class="mi">1</span>


            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;common&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="n">common</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;names of genes&quot;</span><span class="p">]</span><span class="o">=</span><span class="n">query2</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;n_common&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">common</span><span class="p">)</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;query gene info length&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">q1_interact</span><span class="p">)</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;interactors of query gene info length&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">q2_interact</span><span class="p">)</span>
            <span class="k">if</span> <span class="nb">len</span><span class="p">(</span><span class="n">q1</span><span class="p">)</span><span class="o">==</span><span class="mi">0</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query subset&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="mi">0</span>
            <span class="k">else</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query subset&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">d</span><span class="p">)</span><span class="o">/</span><span class="nb">len</span><span class="p">(</span><span class="n">q1_interact</span><span class="p">)</span> <span class="o">*</span><span class="mi">100</span>

            <span class="k">if</span> <span class="nb">len</span><span class="p">(</span><span class="n">q2</span><span class="p">)</span><span class="o">==</span><span class="mi">0</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query 2 subset  &quot;</span><span class="p">]</span> <span class="o">=</span> <span class="mi">0</span>
            <span class="k">else</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query 2 subset  &quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">d</span><span class="p">)</span><span class="o">/</span><span class="nb">len</span><span class="p">(</span><span class="n">q2_interact</span><span class="p">)</span> <span class="o">*</span><span class="mi">100</span>

            <span class="n">q1_filt</span><span class="o">=</span><span class="n">q1</span><span class="p">[</span><span class="n">q1</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">3</span><span class="p">]]</span><span class="o">==</span><span class="n">query2</span><span class="p">]</span>





    <span class="c1">#        if q1_filt[col_label[4]].any()==&#39;Synthetic Lethality &#39;:</span>
            <span class="n">interactions_lethality</span><span class="o">=</span><span class="p">(</span><span class="n">q1_filt</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">4</span><span class="p">]]</span><span class="o">==</span><span class="s1">&#39;Synthetic Lethality&#39;</span><span class="p">)</span>
            <span class="n">interactions_negative</span><span class="o">=</span><span class="p">(</span><span class="n">q1_filt</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">4</span><span class="p">]]</span><span class="o">==</span><span class="s1">&#39;Negative Genetic&#39;</span><span class="p">)</span>
            <span class="n">interactions_positive</span><span class="o">=</span><span class="p">(</span><span class="n">q1_filt</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">4</span><span class="p">]]</span><span class="o">==</span><span class="s1">&#39;Positive Genetic&#39;</span><span class="p">)</span>

            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;Type&quot;</span><span class="p">]</span><span class="o">=</span> <span class="s1">&#39;not clear interaction&#39;</span>

            <span class="k">if</span> <span class="n">interactions_lethality</span><span class="o">.</span><span class="n">any</span><span class="p">()</span><span class="o">==</span><span class="kc">True</span><span class="p">:</span>

                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;Type&quot;</span><span class="p">]</span><span class="o">=</span> <span class="s1">&#39;Synthetic Lethality&#39;</span>
            <span class="c1">#else:</span>
           <span class="c1"># if interactions_lethality.all()==False and interactions_negative.any()==False and interactions_positive.any()==False:</span>
                <span class="c1">#d2[query1,query2][&quot;Type&quot;]=&#39;Not clear&#39;</span>
            <span class="k">if</span> <span class="n">interactions_negative</span><span class="o">.</span><span class="n">all</span><span class="p">()</span><span class="o">==</span><span class="kc">True</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;Type&quot;</span><span class="p">]</span><span class="o">=</span> <span class="s1">&#39;Negative&#39;</span>
            <span class="c1">#else:</span>
           <span class="c1"># if interactions_lethality.all()==False and interactions_negative.any()==False and interactions_positive.any()==False:</span>
                <span class="c1">#d2[query1,query2][&quot;Type&quot;]=&#39;Not clear&#39;</span>
            <span class="k">if</span> <span class="n">interactions_positive</span><span class="o">.</span><span class="n">all</span><span class="p">()</span><span class="o">==</span><span class="kc">True</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;Type&quot;</span><span class="p">]</span><span class="o">=</span> <span class="s1">&#39;Positive&#39;</span>
            <span class="c1">#else:</span>
           <span class="c1"># if interactions_lethality.all()==False and interactions_negative.any()==False and interactions_positive.any()==False:</span>
                <span class="c1">#d2[query1,query2][&quot;Type&quot;]=&#39;Not clear&#39;</span>
           <span class="c1"># else:</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;interact_annotation&quot;</span><span class="p">]</span><span class="o">=</span><span class="n">q1_filt</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">4</span><span class="p">]]</span>
        <span class="n">df</span><span class="o">=</span><span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span><span class="n">d2</span><span class="p">)</span><span class="o">.</span><span class="n">T</span>
        <span class="n">df_sorted</span><span class="o">=</span><span class="n">df</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query subset&#39;</span><span class="p">])</span>
        <span class="n">df_sorted</span><span class="o">=</span><span class="n">df_sorted</span><span class="p">[::</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span>
        <span class="n">df_sorted</span><span class="o">.</span><span class="n">to_excel</span><span class="p">(</span><span class="s2">&quot;data_output_&quot;</span><span class="o">+</span> <span class="s2">&quot;&quot;</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">query</span><span class="p">)</span> <span class="o">+</span> <span class="s2">&quot;.xlsx&quot;</span><span class="p">)</span>
        <span class="n">data</span><span class="o">=</span><span class="n">pd</span><span class="o">.</span><span class="n">read_excel</span><span class="p">(</span><span class="sa">r</span><span class="s1">&#39;C:\Users\linigodelacruz\Documents\PhD_2018\Documentation\Calculations\Functions\data_output_&#39;</span><span class="o">+</span> <span class="s2">&quot;&quot;</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">query</span><span class="p">)</span> <span class="o">+</span> <span class="s2">&quot;.xlsx&quot;</span><span class="p">)</span>
    <span class="k">return</span> <span class="n">data</span>
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span>
<span class="n">genes_passed</span><span class="o">=</span><span class="p">[[</span><span class="s1">&#39;NRP1&#39;</span><span class="p">],[</span><span class="s1">&#39;BEM3&#39;</span><span class="p">],[</span><span class="s1">&#39;GPR1&#39;</span><span class="p">],[</span><span class="s1">&#39;ACT1&#39;</span><span class="p">],[</span><span class="s1">&#39;BRE5&#39;</span><span class="p">],[</span><span class="s1">&#39;CDC28&#39;</span><span class="p">],[</span><span class="s1">&#39;HSP82&#39;</span><span class="p">],[</span><span class="s1">&#39;CDC3&#39;</span><span class="p">],[</span><span class="s1">&#39;ADE13&#39;</span><span class="p">],[</span><span class="s1">&#39;SEC14&#39;</span><span class="p">],[</span><span class="s1">&#39;ARP2&#39;</span><span class="p">],[</span><span class="s1">&#39;BIM1&#39;</span><span class="p">],</span>
              <span class="p">[</span><span class="s1">&#39;BEM4&#39;</span><span class="p">],</span> <span class="p">[</span><span class="s1">&#39;BUB3&#39;</span><span class="p">],[</span><span class="s1">&#39;CNB1&#39;</span><span class="p">],[</span><span class="s1">&#39;GIM3&#39;</span><span class="p">],[</span><span class="s1">&#39;LTE1&#39;</span><span class="p">],[</span><span class="s1">&#39;MAD1&#39;</span><span class="p">],[</span><span class="s1">&#39;PHO85&#39;</span><span class="p">],[</span><span class="s1">&#39;SEC15&#39;</span><span class="p">],[</span><span class="s1">&#39;SEC3&#39;</span><span class="p">],[</span><span class="s1">&#39;SMI1&#39;</span><span class="p">],[</span><span class="s1">&#39;SMY1&#39;</span><span class="p">],[</span><span class="s1">&#39;SWI4&#39;</span><span class="p">],</span>
              <span class="p">[</span><span class="s1">&#39;BEM2&#39;</span><span class="p">],[</span><span class="s1">&#39;BEM1&#39;</span><span class="p">],</span> <span class="p">[</span><span class="s1">&#39;CDC24&#39;</span><span class="p">],[</span><span class="s1">&#39;CDC42&#39;</span><span class="p">],[</span><span class="s1">&#39;RDI1&#39;</span><span class="p">],[</span><span class="s1">&#39;CLA4&#39;</span><span class="p">],[</span><span class="s1">&#39;RSR1&#39;</span><span class="p">],[</span><span class="s1">&#39;RGA1&#39;</span><span class="p">],[</span><span class="s1">&#39;BNI1&#39;</span><span class="p">],[</span><span class="s1">&#39;GIC2&#39;</span><span class="p">],[</span><span class="s1">&#39;BNR1&#39;</span><span class="p">],</span>
              <span class="p">[</span><span class="s1">&#39;MYO1&#39;</span><span class="p">],[</span><span class="s1">&#39;MSB3&#39;</span><span class="p">],[</span><span class="s1">&#39;EXO70&#39;</span><span class="p">],[</span><span class="s1">&#39;CLN1&#39;</span><span class="p">],[</span><span class="s1">&#39;CLN2&#39;</span><span class="p">],[</span><span class="s1">&#39;IRA2&#39;</span><span class="p">],[</span><span class="s1">&#39;WHI2&#39;</span><span class="p">],[</span><span class="s1">&#39;WHI3&#39;</span><span class="p">]]</span>




</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">data_test</span><span class="o">=</span><span class="p">[]</span>
<span class="k">for</span> <span class="n">names</span> <span class="ow">in</span> <span class="n">genes_passed</span><span class="p">:</span>
    <span class="n">output</span><span class="o">=</span><span class="n">query_interactions</span><span class="p">(</span><span class="n">names</span><span class="p">)</span>
    <span class="n">data_test</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">output</span><span class="p">)</span>

</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="c1">## Functions</span>

<span class="k">def</span> <span class="nf">positive_int</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Positive&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query subset&#39;</span><span class="p">]</span>
<span class="k">def</span> <span class="nf">positive_query2</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Positive&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query 2 subset  &#39;</span><span class="p">]</span>
<span class="k">def</span> <span class="nf">negative_int</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Negative&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query subset&#39;</span><span class="p">]</span>
<span class="k">def</span> <span class="nf">negative_query2</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Negative&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query 2 subset  &#39;</span><span class="p">]</span>
<span class="k">def</span> <span class="nf">lethal_int</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Synthetic Lethality&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query subset&#39;</span><span class="p">]</span>
<span class="k">def</span> <span class="nf">lethal_query2</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Synthetic Lethality&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query 2 subset  &#39;</span><span class="p">]</span>

</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">data</span><span class="o">=</span><span class="n">data_test</span>
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="c1">## Positive Interactions</span>
<span class="n">data_positive</span><span class="o">=</span><span class="p">[]</span>
<span class="n">data_positive_query2</span><span class="o">=</span><span class="p">[]</span>
<span class="k">for</span> <span class="n">names</span> <span class="ow">in</span> <span class="n">data</span><span class="p">:</span>
    <span class="n">data_positive</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">positive_int</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>
    <span class="n">data_positive_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">positive_query2</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>

<span class="c1">## Negative Interactions</span>
<span class="n">data_negative</span><span class="o">=</span><span class="p">[]</span>
<span class="n">data_negative_query2</span><span class="o">=</span><span class="p">[]</span>

<span class="k">for</span> <span class="n">names</span> <span class="ow">in</span> <span class="n">data</span><span class="p">:</span>
    <span class="n">data_negative</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">negative_int</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>
    <span class="n">data_negative_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">negative_query2</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>

<span class="c1">## Synthetic Lethal interactions</span>
<span class="n">data_lethality</span><span class="o">=</span><span class="p">[]</span>
<span class="n">data_lethality_query2</span><span class="o">=</span><span class="p">[]</span>

<span class="k">for</span> <span class="n">names</span> <span class="ow">in</span> <span class="n">data</span><span class="p">:</span>
    <span class="n">data_lethality</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">lethal_int</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>
    <span class="n">data_lethality_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">lethal_query2</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="nb">len</span><span class="p">(</span><span class="n">data</span><span class="p">)</span>
</pre></div>
</div>
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>43
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="c1">## Average</span>
<span class="n">mean_values_positive</span><span class="o">=</span><span class="p">[]</span>
<span class="n">mean_values_positive_query2</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_positive</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_positive_query2</span><span class="o">=</span><span class="p">[]</span>

<span class="n">mean_values_negative</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_negative</span><span class="o">=</span><span class="p">[]</span>
<span class="n">mean_values_negative_query2</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_negative_query2</span><span class="o">=</span><span class="p">[]</span>

<span class="n">mean_values_lethal</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_lethal</span><span class="o">=</span><span class="p">[]</span>
<span class="n">mean_values_lethal_query2</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_lethal_query2</span><span class="o">=</span><span class="p">[]</span>

<span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="nb">range</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="nb">len</span><span class="p">(</span><span class="n">data</span><span class="p">)):</span>
    <span class="n">mean_values_positive</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_positive</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">mean_values_positive_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_positive_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_positive</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_positive</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_positive_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_positive_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>

    <span class="n">mean_values_negative</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_negative</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_negative</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_negative</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">mean_values_negative_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_negative_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_negative_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_negative_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>

    <span class="n">mean_values_lethal</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_lethality</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_lethal</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_lethality</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">mean_values_lethal_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_lethality_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_lethal_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_lethality_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>

<span class="n">mean_values</span><span class="o">=</span><span class="p">[</span><span class="n">mean_values_positive</span><span class="p">,</span><span class="n">mean_values_negative</span><span class="p">,</span><span class="n">mean_values_lethal</span><span class="p">]</span>
<span class="n">mean_values_query2</span><span class="o">=</span><span class="p">[</span><span class="n">mean_values_positive_query2</span><span class="p">,</span><span class="n">mean_values_negative_query2</span><span class="p">,</span><span class="n">mean_values_lethal_query2</span><span class="p">]</span>

<span class="n">std_values</span><span class="o">=</span><span class="p">[</span><span class="n">std_values_positive</span><span class="p">,</span><span class="n">std_values_negative</span><span class="p">,</span><span class="n">std_values_lethal</span><span class="p">]</span>
<span class="n">std_values_query2</span><span class="o">=</span><span class="p">[</span><span class="n">std_values_positive_query2</span><span class="p">,</span><span class="n">std_values_negative_query2</span><span class="p">,</span><span class="n">std_values_lethal_query2</span><span class="p">]</span>
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="c1">## Violin plot for the averages and std for each gene positive, negative and synthetic lethal interactions</span>
<span class="c1">## Data quality</span>

<span class="c1"># Make a dictionary with one specific color per group:</span>
<span class="n">my_pal</span> <span class="o">=</span> <span class="p">{</span><span class="s2">&quot;#ff9966&quot;</span><span class="p">,</span>  <span class="s1">&#39;#ffff99&#39;</span><span class="p">,</span> <span class="s1">&#39;#00cc99&#39;</span><span class="p">}</span>

<span class="n">fig</span><span class="p">,</span> <span class="n">axes</span> <span class="o">=</span> <span class="n">plt</span><span class="o">.</span><span class="n">subplots</span><span class="p">(</span><span class="mi">2</span><span class="p">,</span> <span class="mi">2</span><span class="p">,</span> <span class="n">figsize</span><span class="o">=</span><span class="p">(</span><span class="mi">10</span><span class="p">,</span><span class="mi">5</span><span class="p">),</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">100</span><span class="p">,</span> <span class="n">sharex</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">sharey</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">subplots_adjust</span><span class="p">(</span><span class="n">bottom</span><span class="o">=</span><span class="mf">0.2</span><span class="p">,</span> <span class="n">right</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">top</span><span class="o">=</span><span class="mf">1.4</span><span class="p">)</span>
<span class="n">fig</span><span class="o">.</span><span class="n">suptitle</span><span class="p">(</span><span class="s1">&#39;Common interactions vs interaction type&#39;</span><span class="p">,</span><span class="n">x</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span><span class="n">y</span><span class="o">=</span><span class="mf">1.05</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">2</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">mean_values</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">palette</span> <span class="o">=</span> <span class="n">my_pal</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">xticks</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">arange</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">3</span><span class="p">),[</span><span class="s1">&#39;positive&#39;</span><span class="p">,</span><span class="s1">&#39;negative&#39;</span><span class="p">,</span><span class="s1">&#39;synthetic lethal&#39;</span><span class="p">])</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;Average of the query gene interactors&#39;</span><span class="p">,{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;8&#39;</span><span class="p">})</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">120</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">2</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">2</span><span class="p">)</span>
<span class="c1">#sns.stripplot(data=mean_values,palette = my_pal)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">mean_values_query2</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">palette</span> <span class="o">=</span> <span class="n">my_pal</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xticks</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">arange</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">3</span><span class="p">),[</span><span class="s1">&#39;positive&#39;</span><span class="p">,</span><span class="s1">&#39;negative&#39;</span><span class="p">,</span><span class="s1">&#39;synthetic lethal&#39;</span><span class="p">])</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">120</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;Average of 2nd pair gene interactors&#39;</span><span class="p">,{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;8&#39;</span><span class="p">})</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">2</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span> <span class="mi">3</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">std_values</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">palette</span> <span class="o">=</span> <span class="n">my_pal</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">xticks</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">arange</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">3</span><span class="p">),[</span><span class="s1">&#39;positive&#39;</span><span class="p">,</span><span class="s1">&#39;negative&#39;</span><span class="p">,</span><span class="s1">&#39;synthetic lethal&#39;</span><span class="p">])</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;Standard deviation &#39;</span><span class="p">,{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;8&#39;</span><span class="p">})</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">120</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">2</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">4</span><span class="p">)</span>
<span class="c1">#sns.stripplot(data=std_values,palette = my_pal)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">std_values_query2</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">palette</span> <span class="o">=</span> <span class="n">my_pal</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xticks</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">arange</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">3</span><span class="p">),[</span><span class="s1">&#39;positive&#39;</span><span class="p">,</span><span class="s1">&#39;negative&#39;</span><span class="p">,</span><span class="s1">&#39;synthetic lethal&#39;</span><span class="p">])</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">120</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;Std of 2nd pair gene interactors&#39;</span><span class="p">,{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;8&#39;</span><span class="p">})</span>

<span class="n">plt</span><span class="o">.</span><span class="n">tight_layout</span><span class="p">()</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;violinplot-average-std-common-interactors-vs-interaction-type.svg&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;svg&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;violinplot-average-std-common-interactors-vs-interaction-type.png&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;png&#39;</span><span class="p">)</span>
</pre></div>
</div>
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>C:\Users\linigodelacruz\AppData\Local\Continuum\anaconda3\lib\site-packages\scipy\stats\stats.py:1713: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  return np.add.reduce(sorted[indexer] * weights, axis=axis) / sumval
</pre></div>
</div>
<p><img alt="" src="../../_images/output_9_1.png" /></p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">fig</span><span class="p">,</span> <span class="n">axes</span> <span class="o">=</span> <span class="n">plt</span><span class="o">.</span><span class="n">subplots</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span> <span class="mi">3</span><span class="p">,</span> <span class="n">figsize</span><span class="o">=</span><span class="p">(</span><span class="mi">10</span><span class="p">,</span><span class="mi">8</span><span class="p">),</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">100</span><span class="p">,</span> <span class="n">sharex</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">sharey</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">fig</span><span class="o">.</span><span class="n">suptitle</span><span class="p">(</span><span class="s1">&#39;Common interactions per interaction type&#39;</span><span class="p">,</span><span class="n">x</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span><span class="n">y</span><span class="o">=</span><span class="mf">1.05</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Positive interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">10</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_positive</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Negative interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">10</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax1</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_negative</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">3</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Synthetic Lethal interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">10</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax2</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_lethality</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">subplots_adjust</span><span class="p">(</span><span class="n">bottom</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">right</span><span class="o">=</span><span class="mf">1.4</span><span class="p">,</span> <span class="n">top</span><span class="o">=</span><span class="mf">1.4</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">tight_layout</span><span class="p">()</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;boxplot-common-interactors-vs-interaction-type.tiff&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;tiff&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;boxplot-common-interactors-vs-interaction-type.png&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;png&#39;</span><span class="p">)</span>
</pre></div>
</div>
<p><img alt="" src="../../_images/output_10_0.png" /></p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">fig</span><span class="p">,</span> <span class="n">axes</span> <span class="o">=</span> <span class="n">plt</span><span class="o">.</span><span class="n">subplots</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span> <span class="mi">3</span><span class="p">,</span> <span class="n">figsize</span><span class="o">=</span><span class="p">(</span><span class="mi">10</span><span class="p">,</span><span class="mi">8</span><span class="p">),</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">100</span><span class="p">,</span> <span class="n">sharex</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">sharey</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">fig</span><span class="o">.</span><span class="n">suptitle</span><span class="p">(</span><span class="s1">&#39;Common interactions per interaction type&#39;</span><span class="p">,</span><span class="n">x</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span><span class="n">y</span><span class="o">=</span><span class="mf">1.05</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Positive interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_positive_query2</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Negative interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax1</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_negative_query2</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">3</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Synthetic Lethal interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax2</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_lethality_query2</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">subplots_adjust</span><span class="p">(</span><span class="n">bottom</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">right</span><span class="o">=</span><span class="mf">1.4</span><span class="p">,</span> <span class="n">top</span><span class="o">=</span><span class="mf">1.4</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">tight_layout</span><span class="p">()</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;boxplot-2nd-pair-common-interactors-vs-interaction-type.tiff&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;tiff&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;boxplot-2nd-pair-common-interactors-vs-interaction-type.png&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;png&#39;</span><span class="p">)</span>
</pre></div>
</div>
<p><img alt="" src="../../_images/output_11_0.png" /></p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="c1">## Plots histogram</span>

<span class="n">fig</span><span class="p">,</span> <span class="n">axes</span> <span class="o">=</span> <span class="n">plt</span><span class="o">.</span><span class="n">subplots</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span> <span class="mi">3</span><span class="p">,</span> <span class="n">figsize</span><span class="o">=</span><span class="p">(</span><span class="mi">10</span><span class="p">,</span><span class="mi">5</span><span class="p">),</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">100</span><span class="p">,</span> <span class="n">sharex</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">sharey</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">fig</span><span class="o">.</span><span class="n">suptitle</span><span class="p">(</span><span class="s1">&#39;Common interactions vs interaction type&#39;</span><span class="p">,</span><span class="n">x</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span><span class="n">y</span><span class="o">=</span><span class="mf">1.05</span><span class="p">)</span>
<span class="n">kwargs_positive</span> <span class="o">=</span> <span class="nb">dict</span><span class="p">(</span><span class="n">alpha</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">bins</span><span class="o">=</span><span class="mi">20</span><span class="p">,</span> <span class="n">density</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">stacked</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">kwargs_negative</span> <span class="o">=</span> <span class="nb">dict</span><span class="p">(</span><span class="n">alpha</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">bins</span><span class="o">=</span><span class="mi">20</span><span class="p">,</span> <span class="n">density</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">stacked</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">kwargs_lethal</span> <span class="o">=</span> <span class="nb">dict</span><span class="p">(</span><span class="n">alpha</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">bins</span><span class="o">=</span><span class="mi">20</span><span class="p">,</span> <span class="n">density</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">stacked</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>

<span class="c1"># Set the font dictionaries (for plot title and axis titles)</span>
<span class="n">title_font</span> <span class="o">=</span> <span class="p">{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;16&#39;</span><span class="p">,</span> <span class="s1">&#39;color&#39;</span><span class="p">:</span><span class="s1">&#39;black&#39;</span><span class="p">,</span> <span class="s1">&#39;weight&#39;</span><span class="p">:</span><span class="s1">&#39;normal&#39;</span><span class="p">,</span>
              <span class="s1">&#39;verticalalignment&#39;</span><span class="p">:</span><span class="s1">&#39;bottom&#39;</span><span class="p">}</span> <span class="c1"># Bottom vertical alignment for more space</span>
<span class="n">axis_font</span> <span class="o">=</span> <span class="p">{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;8&#39;</span><span class="p">}</span>

<span class="c1">## Positive Interactions</span>
<span class="n">plt</span><span class="o">.</span><span class="n">subplots_adjust</span><span class="p">(</span><span class="n">bottom</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">right</span><span class="o">=</span><span class="mf">1.4</span><span class="p">,</span> <span class="n">top</span><span class="o">=</span><span class="mf">1.4</span><span class="p">)</span>
<span class="c1">#fig.suptitle(&#39;This is a somewhat long figure title&#39;, fontsize=16)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;Normalized frequency&#39;</span><span class="p">,</span><span class="o">**</span><span class="n">axis_font</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlabel</span><span class="p">(</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f common interactions with the query gene&#39;</span><span class="p">,</span><span class="o">**</span><span class="n">axis_font</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mf">0.2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">hist</span><span class="p">(</span><span class="n">data_positive_query2</span><span class="p">[</span><span class="mi">1</span><span class="p">:</span><span class="nb">len</span><span class="p">(</span><span class="n">data_positive_query2</span><span class="p">)],</span><span class="o">**</span><span class="n">kwargs_positive</span><span class="p">,</span><span class="n">label</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;Positive Interactions&#39;</span><span class="p">],</span><span class="n">cumulative</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Positive Interactions&#39;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="c1">#plt.ylabel(&#39;Normalized frequency&#39;,**axis_font)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mf">0.2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlabel</span><span class="p">(</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f common interactions with the query gene&#39;</span><span class="p">,</span><span class="o">**</span><span class="n">axis_font</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">hist</span><span class="p">(</span><span class="n">data_negative_query2</span><span class="p">[</span><span class="mi">1</span><span class="p">:</span><span class="nb">len</span><span class="p">(</span><span class="n">data_negative_query2</span><span class="p">)],</span><span class="o">**</span><span class="n">kwargs_negative</span><span class="p">,</span><span class="n">label</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;Negative Interactions&#39;</span><span class="p">],</span><span class="n">cumulative</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Negative Interactions&#39;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">3</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="c1">#plt.ylabel(&#39;Normalized frequency&#39;,**axis_font)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mf">0.2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlabel</span><span class="p">(</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f common interactions with the query gene&#39;</span><span class="p">,</span><span class="o">**</span><span class="n">axis_font</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">hist</span><span class="p">(</span><span class="n">data_lethality_query2</span><span class="p">[</span><span class="mi">1</span><span class="p">:</span><span class="nb">len</span><span class="p">(</span><span class="n">data_lethality_query2</span><span class="p">)],</span><span class="o">**</span><span class="n">kwargs_lethal</span><span class="p">,</span><span class="n">label</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;Lethality Interactions&#39;</span><span class="p">],</span><span class="n">cumulative</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Synthetic Lethal Interactions&#39;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">tight_layout</span><span class="p">()</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;common-interactors-vs-interaction-type.tiff&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;tiff&#39;</span><span class="p">)</span>
</pre></div>
</div>
<p><img alt="" src="../../_images/output_12_0.png" /></p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">fig</span><span class="p">,</span> <span class="n">axes</span> <span class="o">=</span> <span class="n">plt</span><span class="o">.</span><span class="n">subplots</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span> <span class="mi">3</span><span class="p">,</span> <span class="n">figsize</span><span class="o">=</span><span class="p">(</span><span class="mi">10</span><span class="p">,</span><span class="mi">10</span><span class="p">),</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">100</span><span class="p">,</span> <span class="n">sharex</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">sharey</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">fig</span><span class="o">.</span><span class="n">suptitle</span><span class="p">(</span><span class="s1">&#39;Common interactions per interaction type&#39;</span><span class="p">,</span><span class="n">x</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span><span class="n">y</span><span class="o">=</span><span class="mf">1.01</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Positive interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_positive</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s1">&#39;h&#39;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Negative interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax1</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_negative</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s1">&#39;h&#39;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">3</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Synthetic Lethal interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax2</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_lethality</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s1">&#39;h&#39;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">subplots_adjust</span><span class="p">(</span><span class="n">bottom</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">right</span><span class="o">=</span><span class="mf">1.4</span><span class="p">,</span> <span class="n">top</span><span class="o">=</span><span class="mf">1.4</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">tight_layout</span><span class="p">()</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;violinplot-common-interactors-vs-interaction-type.tiff&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;tiff&#39;</span><span class="p">)</span>
</pre></div>
</div>
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>C:\Users\linigodelacruz\AppData\Local\Continuum\anaconda3\lib\site-packages\scipy\stats\stats.py:1713: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  return np.add.reduce(sorted[indexer] * weights, axis=axis) / sumval
</pre></div>
</div>
<p><img alt="" src="../../_images/output_13_1.png" /></p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span>
</pre></div>
</div>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./journal/2019-08"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            </main>
            <footer class="footer-article noprint">
                
    <!-- Previous / next buttons -->
<div class='prev-next-area'>
    <a class='left-prev' id="prev-link" href="2019-08-22-Exp_Mating-liquid-ylic133_5-yEK7a.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title"><span class="section-number">68. </span>Title: <font color='#ee4466'> 22082019 Mating using liquid drops of yEK7a and ylic133_5 :broken_heart:</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="2019-08-12-Checking-clones-ura-and-ade-growth.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title"><span class="section-number">70. </span>Title : 12082019-Checking clones of Byk832 and ylic33_1 for SATAY :ok:</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            </footer>
        </div>
    </div>
    <div class="footer-content row">
        <footer class="col footer"><p>
  
    By Leila I√±igo de la Cruz<br/>
  
      &copy; Copyright Creative Commons.<br/>
</p>
        </footer>
    </div>
    
</div>


      </div>
    </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf"></script>


  </body>
</html>