
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>4. Title : 16082019 Script for the visualization of the correlation between number of common interactions and type of interactions üßê &#8212; EXPERIMENTAL JOURNAL PHD</title>
    
  <link rel="stylesheet" href="../../_static/css/index.73d71520a4ca3b99cfee5594769eaaae.css">

    
  <link rel="stylesheet"
    href="../../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    
      
  <link rel="stylesheet"
    href="../../_static/vendor/open-sans_all/1.44.1/index.css">
  <link rel="stylesheet"
    href="../../_static/vendor/lato_latin-ext/1.44.1/index.css">

    
    <link rel="stylesheet" href="../../_static/sphinx-book-theme.2d2078699c18a0efb88233928e1cf6ed.css" type="text/css" />
    <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx_tabs/semantic-ui-2.4.1/segment.min.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx_tabs/semantic-ui-2.4.1/menu.min.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx_tabs/semantic-ui-2.4.1/tab.min.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx_tabs/tabs.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/panels-main.c949a650a448cc0ae9fd3441c0e17fb0.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/panels-variables.06eb56fa6e07937060861dad626602ad.css" />
    
  <link rel="preload" as="script" href="../../_static/js/index.3da636dd464baa7582d2.js">

    <script id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
    <script src="../../_static/jquery.js"></script>
    <script src="../../_static/underscore.js"></script>
    <script src="../../_static/doctools.js"></script>
    <script src="../../_static/language_data.js"></script>
    <script src="../../_static/togglebutton.js"></script>
    <script src="../../_static/clipboard.min.js"></script>
    <script src="../../_static/copybutton.js"></script>
    <script async="async" kind="hypothesis" src="https://hypothes.is/embed.js"></script>
    <script kind="utterances">

    var commentsRunWhenDOMLoaded = cb => {
    if (document.readyState != 'loading') {
        cb()
    } else if (document.addEventListener) {
        document.addEventListener('DOMContentLoaded', cb)
    } else {
        document.attachEvent('onreadystatechange', function() {
        if (document.readyState == 'complete') cb()
        })
    }
}

var addUtterances = () => {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src = "https://utteranc.es/client.js";
    script.async = "async";

    script.setAttribute("repo", "leilaicruz/Experimental-journal-jupyterBOOK");
    script.setAttribute("issue-term", "pathname");
    script.setAttribute("theme", "github-light");
    script.setAttribute("label", "üí¨ comment");
    script.setAttribute("crossorigin", "anonymous");

    sections = document.querySelectorAll("div.section");
    if (sections !== null) {
        section = sections[sections.length-1];
        section.appendChild(script);
    }
}
commentsRunWhenDOMLoaded(addUtterances);
</script>
    <script >var togglebuttonSelector = '.toggle, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../../_static/sphinx-book-theme.be0a4a0c39cd630af62a2fcf693f3f06.js"></script>
    <script async="async" src="https://unpkg.com/thebelab@latest/lib/index.js"></script>
    <script >
        const thebe_selector = ".thebe"
        const thebe_selector_input = "pre"
        const thebe_selector_output = ".output"
    </script>
    <script async="async" src="../../_static/sphinx-thebe.js"></script>
    <link rel="shortcut icon" href="../../_static/flowers.jpg"/>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="5. Title: 22082019 Mating using liquid drops of yEK7a and ylic133_5 üíî" href="2019-08-22-Exp_Mating-liquid-ylic133_5-yEK7a.html" />
    <link rel="prev" title="3. Title : 12082019-Checking clones of Byk832 and ylic33_1 for SATAY :üòä" href="2019-08-12-Checking-clones-ura-and-ade-growth.html" />

    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="en" />



  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="80">
    

    <div class="container-xl">
      <div class="row">
          
<div class="col-12 col-md-3 bd-sidebar site-navigation show" id="site-navigation">
    
        <div class="navbar-brand-box">
<a class="navbar-brand text-wrap" href="../../index.html">
  
  <img src="../../_static/flowers.jpg" class="logo" alt="logo">
  
  
  <h1 class="site-logo" id="site-title">EXPERIMENTAL JOURNAL PHD</h1>
  
</a>
</div>

<form class="bd-search d-flex align-items-center" action="../../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form>

<nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
  <ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../../intro.html">
   Experimental progress ü•º ü¶†
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-05-2019
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-05/2019-05-20_Exp_URA_TRANSFORMATION.html">
   1. Title: URA transformation in ADE2 locus ü§î
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-06-2019
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-03_Data_Sequencing_ade2_yll3a.html">
   1. Title: 03062019-Sequencing the homology arms of yll3a strain around the ade2 gene üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-03_Exp_new%20primers_ade2_locus.html">
   2. Title: 03062019-Design of new primers, for the right homology arm, for transformation that avoid the overlap with the RGA1 gene. üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-06_Exp_URA_Transformation.html">
   3. Title: 06062019-Yeast transformation with the OEP1 üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-17_Exp_URA_kickout_transf.html">
   4. Title: 14062019-ylic132  transformation with the OEP2 (unsuccesful)  ‚òπÔ∏è
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-20_Exp_Byk832_transformation_pbk549.html">
   5. Title: 19062019-Byk832 (ETH strain) transformation with pbK549 (very low effciency) ‚òπÔ∏è
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-25_SATAY_card.html">
   6. 25062019-Experimental Card about the SATAY experiment  üßê
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-26_Data_Sequencing_ylic132.html">
   7. Title: 26062019-Sequencing of the URA integration in ade2 locus in ylic132 üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-28_Exp_URA_kickout_transf.html">
   8. Title: 26062019-ylic132  transformation with the OEP2 (unsuccesful) ‚òπÔ∏è
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-07-2019
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-04_Exp_pBK549_digestion_plasmid.html">
   1. Title : 03072019-Digestion pattern of pBK549 (Succesful) üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-08-Data-ylic132_biolog-replicates-sequencing.html">
   2. Title: 26062019-Sequencing of the URA integration in ade2 locus in ylic132 üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-10_Byk832-transformation-pBk549.html">
   3. Title : 10072019-Byk832 transformation using pBK549 plasmid üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-11_Exp_URA-kickout-transformation-ylic132_1_2.html">
   4. Title : 11072019-ylic132-1 and ylic132-2  transformation with the OEP2 üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-16_common-interactor-genes-workflow.html">
   5. 16-07-2019-Script in python to find the most connected genes to a query gene. üßê
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-16-Byk832-checking-clones-ade-ura.html">
   6. Title : 15072019-Check of growth in -ade and -ura of 10 clones of Byk832+pbk549 üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-17_Exp-Genomic-prep-ylic133-clones.html">
   7. Title : 17072019-DNA extraction of ylic133: ura3- ade2- clones anad PCR üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-19_Sequencing-ylic133-biol-replicates.html">
   8. Title: 22072019-Sequencing of ylic133 biological replicates üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-19-glycerol-stocks-ylic133-bio-replicates.html">
   9. Title : 23072019-Glycerols stocks of ylic133 biological replicates
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-23-SATAY-Byk832-1st-attempt.html">
   10. Title: 29072019-SATAY 1st attempt with Byk832-T üòµ
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-24-Strategy-to-SATAY-target-strains.html">
   11. Title : 24072019-Strategies to generate the strains I want to study using SATAY üßê
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-31-Extra-checking-steps-SATAY-Byk832T.html">
   12. Title: 31072019 Double check Byk832T biolog.  replicates suitability for SATAY. üòî
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-08-2019
 </span>
</p>
<ul class="current nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="2019-08-05_Exp-Byk832-transformation-with-pBK549.html">
   1. Title: 05082019-Byk832 transformation with pBk549 üòä
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-08-07-Exp-crossing-plate-yEK7a-and-ylic133_1.html">
   2. Title : 07082019-Crossing of yEK7a with ylic133_1 üíî
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-08-12-Checking-clones-ura-and-ade-growth.html">
   3. Title : 12082019-Checking clones of Byk832 and ylic33_1 for SATAY :üòä
  </a>
 </li>
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   4. Title : 16082019 Script for the visualization of the correlation between number of common interactions and type of interactions üßê
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-08-22-Exp_Mating-liquid-ylic133_5-yEK7a.html">
   5. Title: 22082019 Mating using liquid drops of yEK7a and ylic133_5 üíî
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-08-27-Sporulation-protocol-ylic133%2ByEK7a.html">
   6. Title: 27082019-Sporulation protocol of 8 re-streakings from ylic133_5+yEk7a üòµ
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-09-2019
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-12-ylic133_2%263-transformation-pLL112.html">
   1. Title : 12092019-Transformation of ylic133 with the pLL112 plasmid  üòµ
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-12-ylic133_2%263-transformation-pLL112.html#objective">
   2. Objective
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-24-pBK549-transformation-in-ylic133-and-Byk832.html">
   3. Title : 24092019- pBK549 transformation on ylic133 for further sanity check in SATAY. üòµ
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-24-ylic134-mating-yEK7a.html">
   4. Title :  24092019-Mating of ylic134 with yEK7a üíî
  </a>
 </li>
</ul>

</nav>

 <!-- To handle the deprecated key -->

<div class="navbar_extra_footer">
  Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
</div>

</div>


          


          
<main class="col py-md-3 pl-md-4 bd-content overflow-auto" role="main">
    
    <div class="row topbar fixed-top container-xl">
    <div class="col-12 col-md-3 bd-topbar-whitespace site-navigation show">
    </div>
    <div class="col pl-2 topbar-main">
        
        <button id="navbar-toggler" class="navbar-toggler ml-0" type="button" data-toggle="collapse"
            data-toggle="tooltip" data-placement="bottom" data-target=".site-navigation" aria-controls="navbar-menu"
            aria-expanded="true" aria-label="Toggle navigation" aria-controls="site-navigation"
            title="Toggle navigation" data-toggle="tooltip" data-placement="left">
            <i class="fas fa-bars"></i>
            <i class="fas fa-arrow-left"></i>
            <i class="fas fa-arrow-up"></i>
        </button>
        
        <div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn" aria-label="Download this page"><i
            class="fas fa-download"></i></button>

    
    <div class="dropdown-buttons">
        <!-- ipynb file if we had a myst markdown file -->
        
        <!-- Download raw file -->
        <a class="dropdown-buttons" href="../../_sources/journal/2019-08/2019-08-16-some-plots-gene_int-data.md"><button type="button"
                class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
                data-placement="left">.md</button></a>
        <!-- Download PDF via print -->
        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
            onClick="window.print()" data-toggle="tooltip" data-placement="left">.pdf</button>
    </div>
    
</div>
        <!-- Source interaction buttons -->

<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn"
        aria-label="Connect with source repository"><i class="fab fa-github"></i></button>
    <div class="dropdown-buttons sourcebuttons">
        <a class="repository-button"
            href="https://github.com/leilaicruz/Experimental-journal-jupyterBOOK"><button type="button" class="btn btn-secondary topbarbtn"
                data-toggle="tooltip" data-placement="left" title="Source repository"><i
                    class="fab fa-github"></i>repository</button></a>
        <a class="issues-button"
            href="https://github.com/leilaicruz/Experimental-journal-jupyterBOOK/issues/new?title=Issue%20on%20page%20%2Fjournal/2019-08/2019-08-16-some-plots-gene_int-data.html&body=Your%20issue%20content%20here."><button
                type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip" data-placement="left"
                title="Open an issue"><i class="fas fa-lightbulb"></i>open issue</button></a>
        
    </div>
</div>


        <!-- Full screen (wrap in <a> to have style consistency -->
        <a class="full-screen-button"><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
                data-placement="bottom" onclick="toggleFullScreen()" aria-label="Fullscreen mode"
                title="Fullscreen mode"><i
                    class="fas fa-expand"></i></button></a>

        <!-- Launch buttons -->

    </div>

    <!-- Table of contents -->
    <div class="d-none d-md-block col-md-2 bd-toc show">
        
    </div>
</div>
    <div id="main-content" class="row">
        <div class="col-12 col-md-9 pl-md-3 pr-md-0">
        
              <div>
                
  <div class="section" id="title-16082019-script-for-the-visualization-of-the-correlation-between-number-of-common-interactions-and-type-of-interactions">
<h1><span class="section-number">4. </span>Title : 16082019 Script for the visualization of the correlation between number of common interactions and type of interactions üßê<a class="headerlink" href="#title-16082019-script-for-the-visualization-of-the-correlation-between-number-of-common-interactions-and-type-of-interactions" title="Permalink to this headline">¬∂</a></h1>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="kn">import</span> <span class="nn">pandas</span> <span class="k">as</span> <span class="nn">pd</span>
<span class="kn">import</span> <span class="nn">numpy</span> <span class="k">as</span> <span class="nn">np</span>
<span class="kn">from</span> <span class="nn">collections</span> <span class="kn">import</span> <span class="n">defaultdict</span>
<span class="kn">import</span> <span class="nn">seaborn</span> <span class="k">as</span> <span class="nn">sns</span>
<span class="kn">import</span> <span class="nn">matplotlib.pyplot</span> <span class="k">as</span> <span class="nn">plt</span>
<span class="kn">from</span> <span class="nn">statistics</span> <span class="kn">import</span> <span class="n">mean</span>
<span class="o">%</span><span class="n">matplotlib</span> <span class="n">inline</span>
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="k">def</span> <span class="nf">query_interactions</span><span class="p">(</span><span class="n">query</span><span class="p">):</span>
    <span class="n">d2</span> <span class="o">=</span> <span class="n">defaultdict</span><span class="p">(</span><span class="nb">dict</span><span class="p">)</span>
    <span class="n">data</span><span class="o">=</span><span class="n">pd</span><span class="o">.</span><span class="n">read_excel</span><span class="p">(</span><span class="sa">r</span><span class="s1">&#39;C:\Users\linigodelacruz\Documents\PhD_2018\Documentation\Calculations\data_sgd\Interaction_data_sgd_downloads.xlsx&#39;</span><span class="p">,</span><span class="n">header</span><span class="o">=</span><span class="mi">17</span><span class="p">,</span><span class="n">encoding</span><span class="o">=</span><span class="s2">&quot;utf-8-sig&quot;</span><span class="p">)</span>
    <span class="n">col_label</span><span class="o">=</span><span class="n">data</span><span class="o">.</span><span class="n">columns</span><span class="o">.</span><span class="n">values</span>
    <span class="c1">#query=[&#39;CLN1&#39;]</span>
    <span class="c1"># giant for loop</span>
    <span class="n">names1</span> <span class="o">=</span> <span class="n">query</span>
    <span class="n">i</span><span class="o">=-</span><span class="mi">1</span>
    <span class="k">for</span> <span class="n">query1</span> <span class="ow">in</span> <span class="n">names1</span><span class="p">:</span>
        <span class="c1">#filtering the table just for the value of the query</span>
        <span class="n">q1</span> <span class="o">=</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Standard_Gene_Name_(Bait)&#39;</span><span class="p">]</span><span class="o">==</span><span class="n">query1</span><span class="p">]</span>
        <span class="c1">#r=get_row(query1)</span>
        <span class="c1">#df1 = pd.DataFrame(q1, columns=[&quot;excel_number&quot;, &quot;Interactors of query1&quot;])</span>
        <span class="c1">#gb1 = df1.groupby(&quot;Interactors of query1&quot;, sort=False) # the grouping reorganizes the set alphatically, the sort by default is true!!!!!!</span>
        <span class="c1">#gb1_row = df1.groupby(&quot;excel_number&quot;,sort=False)</span>
        <span class="n">q1_interact</span><span class="o">=</span><span class="n">q1</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">3</span><span class="p">]]</span><span class="o">.</span><span class="n">unique</span><span class="p">()</span>
       <span class="c1"># a for loop for all the interactors of query</span>
        <span class="k">for</span> <span class="n">query2</span> <span class="ow">in</span> <span class="n">q1_interact</span><span class="p">:</span>

            <span class="c1">#i+=1</span>
            <span class="c1"># write in a matrix , where the 1st column are the names in q1 and the first row = q1[0]+get_query(q1[0])</span>
            <span class="n">q2</span><span class="o">=</span><span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Standard_Gene_Name_(Bait)&#39;</span><span class="p">]</span><span class="o">==</span><span class="n">query2</span><span class="p">]</span> <span class="c1">#these are get_query(q1[i])</span>

            <span class="n">q2_interact</span><span class="o">=</span><span class="n">q2</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">3</span><span class="p">]]</span><span class="o">.</span><span class="n">unique</span><span class="p">()</span>
            <span class="c1"># how to avoid that df2 overlaps with the next value?</span>
            <span class="c1">#df2 = pd.DataFrame(q2, columns=[&quot;excel raw number of q2&quot;, &quot;Interactors of query2&quot;])</span>
            <span class="c1">#gb2 = df2.groupby(&quot;Interactors of query2&quot;,sort=False)</span>

            <span class="c1">#names3 = [name3 for name3, row2 in gb2] # to visualize what is in the interactors for query 2</span>


            <span class="n">d</span> <span class="o">=</span> <span class="n">defaultdict</span><span class="p">(</span><span class="nb">int</span><span class="p">)</span>
            <span class="n">common</span> <span class="o">=</span> <span class="p">[]</span>

            <span class="k">for</span> <span class="n">name1</span>  <span class="ow">in</span> <span class="n">q2_interact</span><span class="p">:</span>
                <span class="k">if</span> <span class="n">name1</span> <span class="ow">in</span> <span class="n">q1_interact</span><span class="p">:</span> <span class="c1"># if a gene interactor of the query1 is in interactors of query 2</span>
                    <span class="n">common</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">name1</span><span class="p">)</span>
                    <span class="n">d</span><span class="p">[</span><span class="n">name1</span><span class="p">]</span> <span class="o">+=</span> <span class="mi">1</span>


            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;common&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="n">common</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;names of genes&quot;</span><span class="p">]</span><span class="o">=</span><span class="n">query2</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;n_common&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">common</span><span class="p">)</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;query gene info length&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">q1_interact</span><span class="p">)</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;interactors of query gene info length&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">q2_interact</span><span class="p">)</span>
            <span class="k">if</span> <span class="nb">len</span><span class="p">(</span><span class="n">q1</span><span class="p">)</span><span class="o">==</span><span class="mi">0</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query subset&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="mi">0</span>
            <span class="k">else</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query subset&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">d</span><span class="p">)</span><span class="o">/</span><span class="nb">len</span><span class="p">(</span><span class="n">q1_interact</span><span class="p">)</span> <span class="o">*</span><span class="mi">100</span>

            <span class="k">if</span> <span class="nb">len</span><span class="p">(</span><span class="n">q2</span><span class="p">)</span><span class="o">==</span><span class="mi">0</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query 2 subset  &quot;</span><span class="p">]</span> <span class="o">=</span> <span class="mi">0</span>
            <span class="k">else</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query 2 subset  &quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">d</span><span class="p">)</span><span class="o">/</span><span class="nb">len</span><span class="p">(</span><span class="n">q2_interact</span><span class="p">)</span> <span class="o">*</span><span class="mi">100</span>

            <span class="n">q1_filt</span><span class="o">=</span><span class="n">q1</span><span class="p">[</span><span class="n">q1</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">3</span><span class="p">]]</span><span class="o">==</span><span class="n">query2</span><span class="p">]</span>





    <span class="c1">#        if q1_filt[col_label[4]].any()==&#39;Synthetic Lethality &#39;:</span>
            <span class="n">interactions_lethality</span><span class="o">=</span><span class="p">(</span><span class="n">q1_filt</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">4</span><span class="p">]]</span><span class="o">==</span><span class="s1">&#39;Synthetic Lethality&#39;</span><span class="p">)</span>
            <span class="n">interactions_negative</span><span class="o">=</span><span class="p">(</span><span class="n">q1_filt</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">4</span><span class="p">]]</span><span class="o">==</span><span class="s1">&#39;Negative Genetic&#39;</span><span class="p">)</span>
            <span class="n">interactions_positive</span><span class="o">=</span><span class="p">(</span><span class="n">q1_filt</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">4</span><span class="p">]]</span><span class="o">==</span><span class="s1">&#39;Positive Genetic&#39;</span><span class="p">)</span>

            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;Type&quot;</span><span class="p">]</span><span class="o">=</span> <span class="s1">&#39;not clear interaction&#39;</span>

            <span class="k">if</span> <span class="n">interactions_lethality</span><span class="o">.</span><span class="n">any</span><span class="p">()</span><span class="o">==</span><span class="kc">True</span><span class="p">:</span>

                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;Type&quot;</span><span class="p">]</span><span class="o">=</span> <span class="s1">&#39;Synthetic Lethality&#39;</span>
            <span class="c1">#else:</span>
           <span class="c1"># if interactions_lethality.all()==False and interactions_negative.any()==False and interactions_positive.any()==False:</span>
                <span class="c1">#d2[query1,query2][&quot;Type&quot;]=&#39;Not clear&#39;</span>
            <span class="k">if</span> <span class="n">interactions_negative</span><span class="o">.</span><span class="n">all</span><span class="p">()</span><span class="o">==</span><span class="kc">True</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;Type&quot;</span><span class="p">]</span><span class="o">=</span> <span class="s1">&#39;Negative&#39;</span>
            <span class="c1">#else:</span>
           <span class="c1"># if interactions_lethality.all()==False and interactions_negative.any()==False and interactions_positive.any()==False:</span>
                <span class="c1">#d2[query1,query2][&quot;Type&quot;]=&#39;Not clear&#39;</span>
            <span class="k">if</span> <span class="n">interactions_positive</span><span class="o">.</span><span class="n">all</span><span class="p">()</span><span class="o">==</span><span class="kc">True</span><span class="p">:</span>
                <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;Type&quot;</span><span class="p">]</span><span class="o">=</span> <span class="s1">&#39;Positive&#39;</span>
            <span class="c1">#else:</span>
           <span class="c1"># if interactions_lethality.all()==False and interactions_negative.any()==False and interactions_positive.any()==False:</span>
                <span class="c1">#d2[query1,query2][&quot;Type&quot;]=&#39;Not clear&#39;</span>
           <span class="c1"># else:</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;interact_annotation&quot;</span><span class="p">]</span><span class="o">=</span><span class="n">q1_filt</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">4</span><span class="p">]]</span>
        <span class="n">df</span><span class="o">=</span><span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span><span class="n">d2</span><span class="p">)</span><span class="o">.</span><span class="n">T</span>
        <span class="n">df_sorted</span><span class="o">=</span><span class="n">df</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query subset&#39;</span><span class="p">])</span>
        <span class="n">df_sorted</span><span class="o">=</span><span class="n">df_sorted</span><span class="p">[::</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span>
        <span class="n">df_sorted</span><span class="o">.</span><span class="n">to_excel</span><span class="p">(</span><span class="s2">&quot;data_output_&quot;</span><span class="o">+</span> <span class="s2">&quot;&quot;</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">query</span><span class="p">)</span> <span class="o">+</span> <span class="s2">&quot;.xlsx&quot;</span><span class="p">)</span>
        <span class="n">data</span><span class="o">=</span><span class="n">pd</span><span class="o">.</span><span class="n">read_excel</span><span class="p">(</span><span class="sa">r</span><span class="s1">&#39;C:\Users\linigodelacruz\Documents\PhD_2018\Documentation\Calculations\Functions\data_output_&#39;</span><span class="o">+</span> <span class="s2">&quot;&quot;</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">query</span><span class="p">)</span> <span class="o">+</span> <span class="s2">&quot;.xlsx&quot;</span><span class="p">)</span>
    <span class="k">return</span> <span class="n">data</span>
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span>
genes_passed=[[&#39;NRP1&#39;],[&#39;BEM3&#39;],[&#39;GPR1&#39;],[&#39;ACT1&#39;],[&#39;BRE5&#39;],[&#39;CDC28&#39;],[&#39;HSP82&#39;],[&#39;CDC3&#39;],[&#39;ADE13&#39;],[&#39;SEC14&#39;],[&#39;ARP2&#39;],[&#39;BIM1&#39;],
              [&#39;BEM4&#39;], [&#39;BUB3&#39;],[&#39;CNB1&#39;],[&#39;GIM3&#39;],[&#39;LTE1&#39;],[&#39;MAD1&#39;],[&#39;PHO85&#39;],[&#39;SEC15&#39;],[&#39;SEC3&#39;],[&#39;SMI1&#39;],[&#39;SMY1&#39;],[&#39;SWI4&#39;],
              [&#39;BEM2&#39;],[&#39;BEM1&#39;], [&#39;CDC24&#39;],[&#39;CDC42&#39;],[&#39;RDI1&#39;],[&#39;CLA4&#39;],[&#39;RSR1&#39;],[&#39;RGA1&#39;],[&#39;BNI1&#39;],[&#39;GIC2&#39;],[&#39;BNR1&#39;],
              [&#39;MYO1&#39;],[&#39;MSB3&#39;],[&#39;EXO70&#39;],[&#39;CLN1&#39;],[&#39;CLN2&#39;],[&#39;IRA2&#39;],[&#39;WHI2&#39;],[&#39;WHI3&#39;]]



```python
data_test=[]
for names in genes_passed:
    output=query_interactions(names)
    data_test.append(output)

</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="c1">## Functions</span>

<span class="k">def</span> <span class="nf">positive_int</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Positive&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query subset&#39;</span><span class="p">]</span>
<span class="k">def</span> <span class="nf">positive_query2</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Positive&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query 2 subset  &#39;</span><span class="p">]</span>
<span class="k">def</span> <span class="nf">negative_int</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Negative&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query subset&#39;</span><span class="p">]</span>
<span class="k">def</span> <span class="nf">negative_query2</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Negative&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query 2 subset  &#39;</span><span class="p">]</span>
<span class="k">def</span> <span class="nf">lethal_int</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Synthetic Lethality&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query subset&#39;</span><span class="p">]</span>
<span class="k">def</span> <span class="nf">lethal_query2</span><span class="p">(</span><span class="n">data</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Type&#39;</span><span class="p">]</span><span class="o">==</span><span class="s1">&#39;Synthetic Lethality&#39;</span><span class="p">][</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query 2 subset  &#39;</span><span class="p">]</span>

</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="c1">## Positive Interactions</span>
<span class="n">data_positive</span><span class="o">=</span><span class="p">[]</span>
<span class="n">data_positive_query2</span><span class="o">=</span><span class="p">[]</span>
<span class="k">for</span> <span class="n">names</span> <span class="ow">in</span> <span class="n">data</span><span class="p">:</span>
    <span class="n">data_positive</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">positive_int</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>
    <span class="n">data_positive_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">positive_query2</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>

<span class="c1">## Negative Interactions</span>
<span class="n">data_negative</span><span class="o">=</span><span class="p">[]</span>
<span class="n">data_negative_query2</span><span class="o">=</span><span class="p">[]</span>

<span class="k">for</span> <span class="n">names</span> <span class="ow">in</span> <span class="n">data</span><span class="p">:</span>
    <span class="n">data_negative</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">negative_int</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>
    <span class="n">data_negative_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">negative_query2</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>

<span class="c1">## Synthetic Lethal interactions</span>
<span class="n">data_lethality</span><span class="o">=</span><span class="p">[]</span>
<span class="n">data_lethality_query2</span><span class="o">=</span><span class="p">[]</span>

<span class="k">for</span> <span class="n">names</span> <span class="ow">in</span> <span class="n">data</span><span class="p">:</span>
    <span class="n">data_lethality</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">lethal_int</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>
    <span class="n">data_lethality_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">lethal_query2</span><span class="p">(</span><span class="n">names</span><span class="p">))</span>
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="c1">## Average</span>
<span class="n">mean_values_positive</span><span class="o">=</span><span class="p">[]</span>
<span class="n">mean_values_positive_query2</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_positive</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_positive_query2</span><span class="o">=</span><span class="p">[]</span>

<span class="n">mean_values_negative</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_negative</span><span class="o">=</span><span class="p">[]</span>
<span class="n">mean_values_negative_query2</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_negative_query2</span><span class="o">=</span><span class="p">[]</span>

<span class="n">mean_values_lethal</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_lethal</span><span class="o">=</span><span class="p">[]</span>
<span class="n">mean_values_lethal_query2</span><span class="o">=</span><span class="p">[]</span>
<span class="n">std_values_lethal_query2</span><span class="o">=</span><span class="p">[]</span>

<span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="nb">range</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="nb">len</span><span class="p">(</span><span class="n">data</span><span class="p">)):</span>
    <span class="n">mean_values_positive</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_positive</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">mean_values_positive_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_positive_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_positive</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_positive</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_positive_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_positive_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>

    <span class="n">mean_values_negative</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_negative</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_negative</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_negative</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">mean_values_negative_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_negative_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_negative_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_negative_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>

    <span class="n">mean_values_lethal</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_lethality</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_lethal</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_lethality</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">mean_values_lethal_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">mean</span><span class="p">(</span><span class="n">data_lethality_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>
    <span class="n">std_values_lethal_query2</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">std</span><span class="p">(</span><span class="n">data_lethality_query2</span><span class="p">[</span><span class="n">i</span><span class="p">]))</span>

<span class="n">mean_values</span><span class="o">=</span><span class="p">[</span><span class="n">mean_values_positive</span><span class="p">,</span><span class="n">mean_values_negative</span><span class="p">,</span><span class="n">mean_values_lethal</span><span class="p">]</span>
<span class="n">mean_values_query2</span><span class="o">=</span><span class="p">[</span><span class="n">mean_values_positive_query2</span><span class="p">,</span><span class="n">mean_values_negative_query2</span><span class="p">,</span><span class="n">mean_values_lethal_query2</span><span class="p">]</span>

<span class="n">std_values</span><span class="o">=</span><span class="p">[</span><span class="n">std_values_positive</span><span class="p">,</span><span class="n">std_values_negative</span><span class="p">,</span><span class="n">std_values_lethal</span><span class="p">]</span>
<span class="n">std_values_query2</span><span class="o">=</span><span class="p">[</span><span class="n">std_values_positive_query2</span><span class="p">,</span><span class="n">std_values_negative_query2</span><span class="p">,</span><span class="n">std_values_lethal_query2</span><span class="p">]</span>
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="c1">## Violin plot for the averages and std for each gene positive, negative and synthetic lethal interactions</span>
<span class="c1">## Data quality</span>

<span class="c1"># Make a dictionary with one specific color per group:</span>
<span class="n">my_pal</span> <span class="o">=</span> <span class="p">{</span><span class="s2">&quot;#ff9966&quot;</span><span class="p">,</span>  <span class="s1">&#39;#ffff99&#39;</span><span class="p">,</span> <span class="s1">&#39;#00cc99&#39;</span><span class="p">}</span>

<span class="n">fig</span><span class="p">,</span> <span class="n">axes</span> <span class="o">=</span> <span class="n">plt</span><span class="o">.</span><span class="n">subplots</span><span class="p">(</span><span class="mi">2</span><span class="p">,</span> <span class="mi">2</span><span class="p">,</span> <span class="n">figsize</span><span class="o">=</span><span class="p">(</span><span class="mi">10</span><span class="p">,</span><span class="mi">5</span><span class="p">),</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">100</span><span class="p">,</span> <span class="n">sharex</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">sharey</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">subplots_adjust</span><span class="p">(</span><span class="n">bottom</span><span class="o">=</span><span class="mf">0.2</span><span class="p">,</span> <span class="n">right</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">top</span><span class="o">=</span><span class="mf">1.4</span><span class="p">)</span>
<span class="n">fig</span><span class="o">.</span><span class="n">suptitle</span><span class="p">(</span><span class="s1">&#39;Common interactions vs interaction type&#39;</span><span class="p">,</span><span class="n">x</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span><span class="n">y</span><span class="o">=</span><span class="mf">1.05</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">2</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">mean_values</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">palette</span> <span class="o">=</span> <span class="n">my_pal</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">xticks</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">arange</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">3</span><span class="p">),[</span><span class="s1">&#39;positive&#39;</span><span class="p">,</span><span class="s1">&#39;negative&#39;</span><span class="p">,</span><span class="s1">&#39;synthetic lethal&#39;</span><span class="p">])</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;Average of the query gene interactors&#39;</span><span class="p">,{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;8&#39;</span><span class="p">})</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">120</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">2</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">2</span><span class="p">)</span>
<span class="c1">#sns.stripplot(data=mean_values,palette = my_pal)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">mean_values_query2</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">palette</span> <span class="o">=</span> <span class="n">my_pal</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xticks</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">arange</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">3</span><span class="p">),[</span><span class="s1">&#39;positive&#39;</span><span class="p">,</span><span class="s1">&#39;negative&#39;</span><span class="p">,</span><span class="s1">&#39;synthetic lethal&#39;</span><span class="p">])</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">120</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;Average of 2nd pair gene interactors&#39;</span><span class="p">,{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;8&#39;</span><span class="p">})</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">2</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span> <span class="mi">3</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">std_values</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">palette</span> <span class="o">=</span> <span class="n">my_pal</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">xticks</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">arange</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">3</span><span class="p">),[</span><span class="s1">&#39;positive&#39;</span><span class="p">,</span><span class="s1">&#39;negative&#39;</span><span class="p">,</span><span class="s1">&#39;synthetic lethal&#39;</span><span class="p">])</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;Standard deviation &#39;</span><span class="p">,{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;8&#39;</span><span class="p">})</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">120</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">2</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">4</span><span class="p">)</span>
<span class="c1">#sns.stripplot(data=std_values,palette = my_pal)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">std_values_query2</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">palette</span> <span class="o">=</span> <span class="n">my_pal</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xticks</span><span class="p">(</span><span class="n">np</span><span class="o">.</span><span class="n">arange</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">3</span><span class="p">),[</span><span class="s1">&#39;positive&#39;</span><span class="p">,</span><span class="s1">&#39;negative&#39;</span><span class="p">,</span><span class="s1">&#39;synthetic lethal&#39;</span><span class="p">])</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">120</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;Std of 2nd pair gene interactors&#39;</span><span class="p">,{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;8&#39;</span><span class="p">})</span>

<span class="n">plt</span><span class="o">.</span><span class="n">tight_layout</span><span class="p">()</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;violinplot-average-std-common-interactors-vs-interaction-type.svg&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;svg&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;violinplot-average-std-common-interactors-vs-interaction-type.png&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;png&#39;</span><span class="p">)</span>
</pre></div>
</div>
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>C:\Users\linigodelacruz\AppData\Local\Continuum\anaconda3\lib\site-packages\scipy\stats\stats.py:1713: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  return np.add.reduce(sorted[indexer] * weights, axis=axis) / sumval
</pre></div>
</div>
<p><img alt="" src="../../_images/output_9_1.png" /></p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">fig</span><span class="p">,</span> <span class="n">axes</span> <span class="o">=</span> <span class="n">plt</span><span class="o">.</span><span class="n">subplots</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span> <span class="mi">3</span><span class="p">,</span> <span class="n">figsize</span><span class="o">=</span><span class="p">(</span><span class="mi">10</span><span class="p">,</span><span class="mi">8</span><span class="p">),</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">100</span><span class="p">,</span> <span class="n">sharex</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">sharey</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">fig</span><span class="o">.</span><span class="n">suptitle</span><span class="p">(</span><span class="s1">&#39;Common interactions per interaction type&#39;</span><span class="p">,</span><span class="n">x</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span><span class="n">y</span><span class="o">=</span><span class="mf">1.05</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Positive interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">10</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_positive</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Negative interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">10</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax1</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_negative</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">3</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Synthetic Lethal interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">10</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax2</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_lethality</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">subplots_adjust</span><span class="p">(</span><span class="n">bottom</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">right</span><span class="o">=</span><span class="mf">1.4</span><span class="p">,</span> <span class="n">top</span><span class="o">=</span><span class="mf">1.4</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">tight_layout</span><span class="p">()</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;boxplot-common-interactors-vs-interaction-type.tiff&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;tiff&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;boxplot-common-interactors-vs-interaction-type.png&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;png&#39;</span><span class="p">)</span>
</pre></div>
</div>
<p><img alt="" src="../../_images/output_10_0.png" /></p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">fig</span><span class="p">,</span> <span class="n">axes</span> <span class="o">=</span> <span class="n">plt</span><span class="o">.</span><span class="n">subplots</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span> <span class="mi">3</span><span class="p">,</span> <span class="n">figsize</span><span class="o">=</span><span class="p">(</span><span class="mi">10</span><span class="p">,</span><span class="mi">8</span><span class="p">),</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">100</span><span class="p">,</span> <span class="n">sharex</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">sharey</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">fig</span><span class="o">.</span><span class="n">suptitle</span><span class="p">(</span><span class="s1">&#39;Common interactions per interaction type&#39;</span><span class="p">,</span><span class="n">x</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span><span class="n">y</span><span class="o">=</span><span class="mf">1.05</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Positive interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_positive_query2</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Negative interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax1</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_negative_query2</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">3</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Synthetic Lethal interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax2</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">boxplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_lethality_query2</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s2">&quot;h&quot;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">subplots_adjust</span><span class="p">(</span><span class="n">bottom</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">right</span><span class="o">=</span><span class="mf">1.4</span><span class="p">,</span> <span class="n">top</span><span class="o">=</span><span class="mf">1.4</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">tight_layout</span><span class="p">()</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;boxplot-2nd-pair-common-interactors-vs-interaction-type.tiff&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;tiff&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;boxplot-2nd-pair-common-interactors-vs-interaction-type.png&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;png&#39;</span><span class="p">)</span>
</pre></div>
</div>
<p><img alt="" src="../../_images/output_11_0.png" /></p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="c1">## Plots histogram</span>

<span class="n">fig</span><span class="p">,</span> <span class="n">axes</span> <span class="o">=</span> <span class="n">plt</span><span class="o">.</span><span class="n">subplots</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span> <span class="mi">3</span><span class="p">,</span> <span class="n">figsize</span><span class="o">=</span><span class="p">(</span><span class="mi">10</span><span class="p">,</span><span class="mi">5</span><span class="p">),</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">100</span><span class="p">,</span> <span class="n">sharex</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">sharey</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">fig</span><span class="o">.</span><span class="n">suptitle</span><span class="p">(</span><span class="s1">&#39;Common interactions vs interaction type&#39;</span><span class="p">,</span><span class="n">x</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span><span class="n">y</span><span class="o">=</span><span class="mf">1.05</span><span class="p">)</span>
<span class="n">kwargs_positive</span> <span class="o">=</span> <span class="nb">dict</span><span class="p">(</span><span class="n">alpha</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">bins</span><span class="o">=</span><span class="mi">20</span><span class="p">,</span> <span class="n">density</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">stacked</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">kwargs_negative</span> <span class="o">=</span> <span class="nb">dict</span><span class="p">(</span><span class="n">alpha</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">bins</span><span class="o">=</span><span class="mi">20</span><span class="p">,</span> <span class="n">density</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">stacked</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">kwargs_lethal</span> <span class="o">=</span> <span class="nb">dict</span><span class="p">(</span><span class="n">alpha</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">bins</span><span class="o">=</span><span class="mi">20</span><span class="p">,</span> <span class="n">density</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">stacked</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>

<span class="c1"># Set the font dictionaries (for plot title and axis titles)</span>
<span class="n">title_font</span> <span class="o">=</span> <span class="p">{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;16&#39;</span><span class="p">,</span> <span class="s1">&#39;color&#39;</span><span class="p">:</span><span class="s1">&#39;black&#39;</span><span class="p">,</span> <span class="s1">&#39;weight&#39;</span><span class="p">:</span><span class="s1">&#39;normal&#39;</span><span class="p">,</span>
              <span class="s1">&#39;verticalalignment&#39;</span><span class="p">:</span><span class="s1">&#39;bottom&#39;</span><span class="p">}</span> <span class="c1"># Bottom vertical alignment for more space</span>
<span class="n">axis_font</span> <span class="o">=</span> <span class="p">{</span><span class="s1">&#39;fontname&#39;</span><span class="p">:</span><span class="s1">&#39;Arial&#39;</span><span class="p">,</span> <span class="s1">&#39;size&#39;</span><span class="p">:</span><span class="s1">&#39;8&#39;</span><span class="p">}</span>

<span class="c1">## Positive Interactions</span>
<span class="n">plt</span><span class="o">.</span><span class="n">subplots_adjust</span><span class="p">(</span><span class="n">bottom</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">right</span><span class="o">=</span><span class="mf">1.4</span><span class="p">,</span> <span class="n">top</span><span class="o">=</span><span class="mf">1.4</span><span class="p">)</span>
<span class="c1">#fig.suptitle(&#39;This is a somewhat long figure title&#39;, fontsize=16)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;Normalized frequency&#39;</span><span class="p">,</span><span class="o">**</span><span class="n">axis_font</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlabel</span><span class="p">(</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f common interactions with the query gene&#39;</span><span class="p">,</span><span class="o">**</span><span class="n">axis_font</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mf">0.2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">hist</span><span class="p">(</span><span class="n">data_positive_query2</span><span class="p">[</span><span class="mi">1</span><span class="p">:</span><span class="nb">len</span><span class="p">(</span><span class="n">data_positive_query2</span><span class="p">)],</span><span class="o">**</span><span class="n">kwargs_positive</span><span class="p">,</span><span class="n">label</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;Positive Interactions&#39;</span><span class="p">],</span><span class="n">cumulative</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Positive Interactions&#39;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="c1">#plt.ylabel(&#39;Normalized frequency&#39;,**axis_font)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mf">0.2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlabel</span><span class="p">(</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f common interactions with the query gene&#39;</span><span class="p">,</span><span class="o">**</span><span class="n">axis_font</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">hist</span><span class="p">(</span><span class="n">data_negative_query2</span><span class="p">[</span><span class="mi">1</span><span class="p">:</span><span class="nb">len</span><span class="p">(</span><span class="n">data_negative_query2</span><span class="p">)],</span><span class="o">**</span><span class="n">kwargs_negative</span><span class="p">,</span><span class="n">label</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;Negative Interactions&#39;</span><span class="p">],</span><span class="n">cumulative</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Negative Interactions&#39;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">3</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="c1">#plt.ylabel(&#39;Normalized frequency&#39;,**axis_font)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylim</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span><span class="mf">0.2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlabel</span><span class="p">(</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f common interactions with the query gene&#39;</span><span class="p">,</span><span class="o">**</span><span class="n">axis_font</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">hist</span><span class="p">(</span><span class="n">data_lethality_query2</span><span class="p">[</span><span class="mi">1</span><span class="p">:</span><span class="nb">len</span><span class="p">(</span><span class="n">data_lethality_query2</span><span class="p">)],</span><span class="o">**</span><span class="n">kwargs_lethal</span><span class="p">,</span><span class="n">label</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;Lethality Interactions&#39;</span><span class="p">],</span><span class="n">cumulative</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Synthetic Lethal Interactions&#39;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">tight_layout</span><span class="p">()</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;common-interactors-vs-interaction-type.tiff&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;tiff&#39;</span><span class="p">)</span>
</pre></div>
</div>
<p><img alt="" src="../../_images/output_12_0.png" /></p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">fig</span><span class="p">,</span> <span class="n">axes</span> <span class="o">=</span> <span class="n">plt</span><span class="o">.</span><span class="n">subplots</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span> <span class="mi">3</span><span class="p">,</span> <span class="n">figsize</span><span class="o">=</span><span class="p">(</span><span class="mi">10</span><span class="p">,</span><span class="mi">10</span><span class="p">),</span> <span class="n">dpi</span><span class="o">=</span><span class="mi">100</span><span class="p">,</span> <span class="n">sharex</span><span class="o">=</span><span class="kc">True</span><span class="p">,</span> <span class="n">sharey</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">fig</span><span class="o">.</span><span class="n">suptitle</span><span class="p">(</span><span class="s1">&#39;Common interactions per interaction type&#39;</span><span class="p">,</span><span class="n">x</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span><span class="n">y</span><span class="o">=</span><span class="mf">1.01</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Positive interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">ylabel</span><span class="p">(</span><span class="s1">&#39;&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_positive</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s1">&#39;h&#39;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span> <span class="mi">2</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Negative interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax1</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_negative</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s1">&#39;h&#39;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">subplot</span><span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">3</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">title</span><span class="p">(</span><span class="s1">&#39;Synthetic Lethal interactors&#39;</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="kc">True</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlim</span><span class="p">(</span><span class="o">-</span><span class="mi">50</span><span class="p">,</span><span class="mi">100</span><span class="p">)</span>
<span class="n">ax2</span> <span class="o">=</span> <span class="n">sns</span><span class="o">.</span><span class="n">violinplot</span><span class="p">(</span><span class="n">data</span><span class="o">=</span><span class="n">data_lethality</span><span class="p">,</span><span class="n">scale</span><span class="o">=</span><span class="s2">&quot;count&quot;</span><span class="p">,</span><span class="n">orient</span><span class="o">=</span><span class="s1">&#39;h&#39;</span><span class="p">)</span>
<span class="n">sns</span><span class="o">.</span><span class="n">set_style</span><span class="p">(</span><span class="s2">&quot;whitegrid&quot;</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">subplots_adjust</span><span class="p">(</span><span class="n">bottom</span><span class="o">=</span><span class="mf">0.5</span><span class="p">,</span> <span class="n">right</span><span class="o">=</span><span class="mf">1.4</span><span class="p">,</span> <span class="n">top</span><span class="o">=</span><span class="mf">1.4</span><span class="p">)</span>


<span class="n">plt</span><span class="o">.</span><span class="n">tight_layout</span><span class="p">()</span>
<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;violinplot-common-interactors-vs-interaction-type.tiff&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;tiff&#39;</span><span class="p">)</span>
</pre></div>
</div>
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>C:\Users\linigodelacruz\AppData\Local\Continuum\anaconda3\lib\site-packages\scipy\stats\stats.py:1713: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  return np.add.reduce(sorted[indexer] * weights, axis=axis) / sumval
</pre></div>
</div>
<p><img alt="" src="../../_images/output_13_1.png" /></p>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./journal\2019-08"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
        </div>
    </div>
    
    
    <div class='prev-next-bottom'>
        
    <a class='left-prev' id="prev-link" href="2019-08-12-Checking-clones-ura-and-ade-growth.html" title="previous page"><span class="section-number">3. </span>Title : 12082019-Checking clones of Byk832 and ylic33_1 for SATAY :üòä</a>
    <a class='right-next' id="next-link" href="2019-08-22-Exp_Mating-liquid-ylic133_5-yEK7a.html" title="next page"><span class="section-number">5. </span>Title: 22082019 Mating using liquid drops of yEK7a and ylic133_5 üíî</a>

    </div>
    <footer class="footer mt-5 mt-md-0">
    <div class="container">
      <p>
        
          By Leila I√±igo de la Cruz<br/>
        
            &copy; Copyright Creative Commons.<br/>
      </p>
    </div>
  </footer>
</main>


      </div>
    </div>

    
  <script src="../../_static/js/index.3da636dd464baa7582d2.js"></script>


    
  </body>
</html>