
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>1. Title : 20200810-SATAY for \(\Delta\)bem2, \(\Delta\)nrp1 and WT &#8212; EXPERIMENTAL JOURNAL PHD</title>
    
  <link rel="stylesheet" href="../../_static/css/index.73d71520a4ca3b99cfee5594769eaaae.css">

    
  <link rel="stylesheet"
    href="../../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    
      
  <link rel="stylesheet"
    href="../../_static/vendor/open-sans_all/1.44.1/index.css">
  <link rel="stylesheet"
    href="../../_static/vendor/lato_latin-ext/1.44.1/index.css">

    
    <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
    <link rel="stylesheet" href="../../_static/sphinx-book-theme.40e2e510f6b7d1648584402491bb10fe.css" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx_tabs/semantic-ui-2.4.1/segment.min.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx_tabs/semantic-ui-2.4.1/menu.min.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx_tabs/semantic-ui-2.4.1/tab.min.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx_tabs/tabs.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/panels-main.c949a650a448cc0ae9fd3441c0e17fb0.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/panels-variables.06eb56fa6e07937060861dad626602ad.css" />
    
  <link rel="preload" as="script" href="../../_static/js/index.3da636dd464baa7582d2.js">

    <script id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
    <script src="../../_static/jquery.js"></script>
    <script src="../../_static/underscore.js"></script>
    <script src="../../_static/doctools.js"></script>
    <script src="../../_static/language_data.js"></script>
    <script src="../../_static/togglebutton.js"></script>
    <script src="../../_static/clipboard.min.js"></script>
    <script src="../../_static/copybutton.js"></script>
    <script async="async" kind="hypothesis" src="https://hypothes.is/embed.js"></script>
    <script kind="utterances">

    var commentsRunWhenDOMLoaded = cb => {
    if (document.readyState != 'loading') {
        cb()
    } else if (document.addEventListener) {
        document.addEventListener('DOMContentLoaded', cb)
    } else {
        document.attachEvent('onreadystatechange', function() {
        if (document.readyState == 'complete') cb()
        })
    }
}

var addUtterances = () => {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src = "https://utteranc.es/client.js";
    script.async = "async";

    script.setAttribute("repo", "leilaicruz/Experimental-journal-jupyterBOOK");
    script.setAttribute("issue-term", "pathname");
    script.setAttribute("theme", "github-light");
    script.setAttribute("label", "üí¨ comment");
    script.setAttribute("crossorigin", "anonymous");

    sections = document.querySelectorAll("div.section");
    if (sections !== null) {
        section = sections[sections.length-1];
        section.appendChild(script);
    }
}
commentsRunWhenDOMLoaded(addUtterances);
</script>
    <script >var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../../_static/sphinx-book-theme.d31b09fe5c1d09cb49b26a786de4a05d.js"></script>
    <script async="async" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
    <script type="text/x-mathjax-config">MathJax.Hub.Config({"TeX": {"Macros": {"N": "\\mathbb{N}", "floor": ["\\lfloor#1\\rfloor", 1], "bmat": ["\\left[\\begin{array}"], "emat": ["\\end{array}\\right]"]}}, "tex2jax": {"inlineMath": [["\\(", "\\)"]], "displayMath": [["\\[", "\\]"]], "processRefs": false, "processEnvironments": false}})</script>
    <script async="async" src="https://unpkg.com/thebelab@latest/lib/index.js"></script>
    <script >
        const thebe_selector = ".thebe"
        const thebe_selector_input = "pre"
        const thebe_selector_output = ".output"
    </script>
    <script async="async" src="../../_static/sphinx-thebe.js"></script>
    <link rel="shortcut icon" href="../../_static/flowers.jpg"/>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="2. DNA extraction from SATAY samples" href="2020-08-18-DNA-Prep-SATAY.html" />
    <link rel="prev" title="4. Title : Digestion and Ligation for sequencing" href="../2020-10/2020-10-21-Digestion-Ligation2Sequence.html" />

    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="en" />



  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="80">
    

    <div class="container-xl">
      <div class="row">
          
<div class="col-12 col-md-3 bd-sidebar site-navigation show" id="site-navigation">
    
        <div class="navbar-brand-box">
<a class="navbar-brand text-wrap" href="../../index.html">
  
  <img src="../../_static/flowers.jpg" class="logo" alt="logo">
  
  
  <h1 class="site-logo" id="site-title">EXPERIMENTAL JOURNAL PHD</h1>
  
</a>
</div><form class="bd-search d-flex align-items-center" action="../../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form>
<nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
    <ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../../intro.html">
   Experimental progress ü•º ü¶†
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  General docs
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-21-Pandoc-installation.html">
   1. How to set up Pandoc in your PC
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-07-Balance-of-2019.html">
   2. Balance of the 2019 and outlook to the 2020
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-03/2020-03-17-brainstorm-research-questions.html">
   3. Brainstorming for defining the research questions
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-02-Yearly-Progress-Meeting.html">
   4. Title: Yearly Progress Meeting (Starting my 3rd PhD year)
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-06-Jupyter-notebook-sync-to-md.html">
   5. Title: 06062020- Steps to successfully interoperate between Jupyter notebooks and markdown .
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-11-2020
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-11/2020-11-17-gDNA-extraction-and-PCR-with-yll138.html">
   1. Title : gDNA extraction of yll138
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-11/2020-11-27-gDNA-extraction-and-PCR-with-ywt04a.html">
   2. Title : gDNA extraction from yWT04a
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-10-2020
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-14-bem3d-bem3dnrp1d-strain-construction.html">
   1. Title : ylic133 and ylic136 transformation with bem3d::NAT
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-15-More-DNA-extraction-WT-dnrp1.html">
   2. Title: More DNA extraction of dnrp1 and WT strains
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-19-Digestion-Ligation-Test-New-SATAYDNA.html">
   3. Title : Digestion and Ligation Test for new SATAY DNA
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-21-Digestion-Ligation2Sequence.html">
   4. Title : Digestion and Ligation for sequencing
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-08-2020
 </span>
</p>
<ul class="current nav sidenav_l1">
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   1. Title : 20200810-SATAY for
   <span class="math notranslate nohighlight">
    \(\Delta\)
   </span>
   bem2,
   <span class="math notranslate nohighlight">
    \(\Delta\)
   </span>
   nrp1 and WT
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2020-08-18-DNA-Prep-SATAY.html">
   2. DNA extraction from SATAY samples
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2020-08-24-Digestion-circularization-PCR.html">
   3. Title: Towards DNA sequencing: Digestion, Circularization and PCR of the transposon-genome junctions.
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-07-2020
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-07/2020-07-06-Preparation-for-PCR-check-of-BEM2-bem2.html">
   1. Title : Preparation for PCR check of BEM2 presence and absence in strain ylic135
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-07/2020-07-10-Transformation-of-ylic135-and-ylic136-with-pbK549.html">
   2. Title : Transformation of ylic135 and ylic136 with pbK549 for further SATAY experiment
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-07/2020-07-15-Transformation-of-ylic133-with-pBK549-for-SATAY.html">
   3. Title : Transformation of ylic133 (WT ade- ura-) with pbK549 for SATAY
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-06-2020
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-17-Experimental-planing.html">
   1. Experimental Plan from June 2020 -June 2021
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-22-Essential-and-not-essential-differences-in-interactors.html">
   2. How different are the number of interactors of essential genes compared with non essential genes.
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-04-2020
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-08-summary-FACs-experiments.html">
   1. Summary of FACs experiments
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-23-Machine-Learning-study.html">
   2. Title: 23042020-Summary on machine learning study
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-24-plan-of-next-experiments.html">
   3. Title: 24042020-Plan for next experiments
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-29-how-commons-properties-are-useful-to-predict-SL.html">
   4. Title : From common properties to Synthetic Lethality prediction
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-30-Checking-bem2-deletion-on-ylic133_5.html">
   5. Title: Checking bem2 deletion on the transformed strain ylic133_5
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-03-2020
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-03/2020-03-04-FACs.html">
   1. Title : 20200305-FACs experiment with strains ywkd065, ywkd024, ywkd038 and ywkd001
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-02-2020
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-09-dnrp1-dbem2-for-satay.html">
   1. Title : Design of the genomic transformation to make dnrp1 and dbem2 strains usable for SATAY
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-13-Comparison-of%20Werner-FACs-settings-with-mine.html">
   2. Title : Comparison between the fAcs settings used for Werner in 2017 with the ones I am using currently.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-21-hands-on-data-from-facs.html">
   3. Title :What are the output channels from flow cytometry actualling telling us about the fluorescence of every cell?
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-23-ylic133-5-transformations.html">
   4. Title: Transformation of ylic133_5 with the PCR products from yll137 and yll140
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-25-FACs-starting-with-30C-shaking.html">
   5. Title : 20200227-FACs experiment with strains ywkd065, ywkd024, ywkd038 and ywkd001
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-01-2020
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-14-essential-genes-insights.html">
   1. Title:14012020-Mapping essential genes  in WT and getting insights from it.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-07-machine-learning-proposal-projects.html">
   2. Machine learning projects proposal for MsC/BsC/internships
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-20-FACs-as-microscopy-conditions-prep.html">
   3. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pAl1-Cdc42-sfGFP strains. (I)
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-28-FACs-as-microscopy-conditions-prep-II.html">
   4. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pGAl1-Cdc42-sfGFP strains (II).
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-02-04-FACs-as-microscopy-conditions-prep-III.html">
   5. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pAl1-Cdc42-sfGFP strains (III).
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-10-2019
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-01-pBK549-transformation-in-ylic133-clonesII.html">
   1. Title : 01102019- II pBK549 transformation on ylic133 for further sanity check in SATAY üòî
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-03-pBK549-transformation-in-ylic133-clones-III.html">
   2. Title: 03102019- III pBK549 transformation on ylic133 for further sanity check in SATAY
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-09-Sanity-check-ylic133%2BpBk549.html">
   3. Title: 09102019- Sanity check of transformants ylic133+pBk549, Byk832+pBk549 to know if they suitable for SATAY.
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-09-2019
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-12-ylic133_2%263-transformation-pLL112.html">
   1. Title : 12092019-Transformation of ylic133 with the pLL112 plasmid  üòµ
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-24-pBK549-transformation-in-ylic133-and-Byk832.html">
   2. Title : 24092019- pBK549 transformation on ylic133 for further sanity check in SATAY. üòµ
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-24-ylic134-mating-yEK7a.html">
   3. Title :  24092019-Mating of ylic134 with yEK7a üíî
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-08-2019
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-05_Exp-Byk832-transformation-with-pBK549.html">
   1. Title: 05082019-Byk832 transformation with pBk549
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-07-Exp-crossing-plate-yEK7a-and-ylic133_1.html">
   2. Title : 07082019-Crossing of yEK7a with ylic133_1 üíî
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-12-Checking-clones-ura-and-ade-growth.html">
   3. Title : 12082019-Checking clones of Byk832 and ylic33_1 for SATAY
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-16-some-plots-gene_int-data.html">
   4. Title : 16082019 Script for the visualization of the correlation between number of common interactions and type of interactions üßê
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-22-Exp_Mating-liquid-ylic133_5-yEK7a.html">
   5. Title: 22082019 Mating using liquid drops of yEK7a and ylic133_5 üíî
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-27-Sporulation-protocol-ylic133%2ByEK7a.html">
   6. Title: 27082019-Sporulation protocol of 8 re-streakings from ylic133_5+yEk7a üòµ
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-07-2019
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-04_Exp_pBK549_digestion_plasmid.html">
   1. Title : 03072019-Digestion pattern of pBK549 (Successful)
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-08-Data-ylic132_biolog-replicates-sequencing.html">
   2. Title: 26062019-Sequencing of the URA integration in ade2 locus in ylic132
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-10_Byk832-transformation-pBk549.html">
   3. Title : 10072019-Byk832 transformation using pBK549 plasmid
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-11_Exp_URA-kickout-transformation-ylic132_1_2.html">
   4. Title : 11072019-ylic132-1 and ylic132-2  transformation with the OEP2
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-16_common-interactor-genes-workflow.html">
   5. 16-07-2019-Script in python to find the most connected genes to a query gene. üßê
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-16-Byk832-checking-clones-ade-ura.html">
   6. Title : 15072019-Check of growth in -ade and -ura of 10 clones of Byk832+pbk549
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-17_Exp-Genomic-prep-ylic133-clones.html">
   7. Title : 17072019-DNA extraction of ylic133: ura3- ade2- clones anad PCR
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-19_Sequencing-ylic133-biol-replicates.html">
   8. Title: 22072019-Sequencing of ylic133 biological replicates
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-19-glycerol-stocks-ylic133-bio-replicates.html">
   9. Title : 23072019-Glycerols stocks of ylic133 biological replicates
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-23-SATAY-Byk832-1st-attempt.html">
   10. Title: 29072019-SATAY 1st attempt with Byk832-T üòµ
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-24-Strategy-to-SATAY-target-strains.html">
   11. Title : 24072019-Strategies to generate the strains I want to study using SATAY üßê
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-07/2019-07-31-Extra-checking-steps-SATAY-Byk832T.html">
   12. Title: 31072019 Double check Byk832T biolog.  replicates suitability for SATAY. üòî
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-06-2019
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-03_Data_Sequencing_ade2_yll3a.html">
   1. Title: 03062019-Sequencing the homology arms of yll3a strain around the ade2 gene
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-03_Exp_new%20primers_ade2_locus.html">
   2. Title: 03062019-Design of new primers, for the right homology arm, for transformation that avoid the overlap with the RGA1 gene.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-06_Exp_URA_Transformation.html">
   3. Title: 06062019-Yeast transformation with the OEP1
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-17_Exp_URA_kickout_transf.html">
   4. Title: 14062019-ylic132  transformation with the OEP2 (unsuccesful)  ‚òπÔ∏è
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-20_Exp_Byk832_transformation_pbk549.html">
   5. Title: 19062019-Byk832 (ETH strain) transformation with pbK549 (very low effciency) ‚òπÔ∏è
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-25_SATAY_card.html">
   6. 25062019-Experimental Card about the SATAY experiment  üßê
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-26_Data_Sequencing_ylic132.html">
   7. Title: 26062019-Sequencing of the URA integration in ade2 locus in ylic132
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-28_Exp_URA_kickout_transf.html">
   8. Title: 26062019-ylic132  transformation with the OEP2 (unsuccesful) ‚òπÔ∏è
  </a>
 </li>
</ul>
<p class="caption collapsible-parent">
 <span class="caption-text">
  Experimental journal-05-2019
 </span>
</p>
<ul class="nav sidenav_l1">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-05/2019-05-20_Exp_URA_TRANSFORMATION.html">
   1. Title: URA transformation in ADE2 locus ü§î
  </a>
 </li>
</ul>

</nav> <!-- To handle the deprecated key -->

<div class="navbar_extra_footer">
  Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
</div>

</div>


          


          
<main class="col py-md-3 pl-md-4 bd-content overflow-auto" role="main">
    
    <div class="row topbar fixed-top container-xl">
    <div class="col-12 col-md-3 bd-topbar-whitespace site-navigation show">
    </div>
    <div class="col pl-2 topbar-main">
        
        <button id="navbar-toggler" class="navbar-toggler ml-0" type="button" data-toggle="collapse"
            data-toggle="tooltip" data-placement="bottom" data-target=".site-navigation" aria-controls="navbar-menu"
            aria-expanded="true" aria-label="Toggle navigation" aria-controls="site-navigation"
            title="Toggle navigation" data-toggle="tooltip" data-placement="left">
            <i class="fas fa-bars"></i>
            <i class="fas fa-arrow-left"></i>
            <i class="fas fa-arrow-up"></i>
        </button>
        
        
<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn" aria-label="Download this page"><i
            class="fas fa-download"></i></button>

    <div class="dropdown-buttons">
        <!-- ipynb file if we had a myst markdown file -->
        
        <!-- Download raw file -->
        <a class="dropdown-buttons" href="../../_sources/journal/2020-08/2020-08-10-SATAY-bem2d-nrp1d-WT.md"><button type="button"
                class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
                data-placement="left">.md</button></a>
        <!-- Download PDF via print -->
        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
            onClick="window.print()" data-toggle="tooltip" data-placement="left">.pdf</button>
    </div>
</div>

        <!-- Source interaction buttons -->

<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn"
        aria-label="Connect with source repository"><i class="fab fa-github"></i></button>
    <div class="dropdown-buttons sourcebuttons">
        <a class="repository-button"
            href="https://github.com/leilaicruz/Experimental-journal-jupyterBOOK"><button type="button" class="btn btn-secondary topbarbtn"
                data-toggle="tooltip" data-placement="left" title="Source repository"><i
                    class="fab fa-github"></i>repository</button></a>
        <a class="issues-button"
            href="https://github.com/leilaicruz/Experimental-journal-jupyterBOOK/issues/new?title=Issue%20on%20page%20%2Fjournal/2020-08/2020-08-10-SATAY-bem2d-nrp1d-WT.html&body=Your%20issue%20content%20here."><button
                type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip" data-placement="left"
                title="Open an issue"><i class="fas fa-lightbulb"></i>open issue</button></a>
        
    </div>
</div>


        <!-- Full screen (wrap in <a> to have style consistency -->
        <a class="full-screen-button"><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
                data-placement="bottom" onclick="toggleFullScreen()" aria-label="Fullscreen mode"
                title="Fullscreen mode"><i
                    class="fas fa-expand"></i></button></a>

        <!-- Launch buttons -->

    </div>

    <!-- Table of contents -->
    <div class="d-none d-md-block col-md-2 bd-toc show">
        
        <div class="tocsection onthispage pt-5 pb-3">
            <i class="fas fa-list"></i> Contents
        </div>
        <nav id="bd-toc-nav">
            <ul class="nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#date">
   1.1. Date
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#objective">
   1.2. Objective
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#method">
   1.3. Method
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#defining-the-complexity-of-the-library">
     1.3.1. Defining the complexity of the library
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#number-of-cycles-during-the-reseed">
     1.3.2. Number of cycles during the reseed
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#results">
   1.4. Results
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#next-steps">
   1.5. Next steps
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#conclusion">
   1.6. Conclusion
  </a>
 </li>
</ul>

        </nav>
        
    </div>
</div>
    <div id="main-content" class="row">
        <div class="col-12 col-md-9 pl-md-3 pr-md-0">
        
              <div>
                
  <div class="section" id="title-20200810-satay-for-deltabem2-deltanrp1-and-wt">
<h1><span class="section-number">1. </span>Title : 20200810-SATAY for <span class="math notranslate nohighlight">\(\Delta\)</span>bem2, <span class="math notranslate nohighlight">\(\Delta\)</span>nrp1 and WT<a class="headerlink" href="#title-20200810-satay-for-deltabem2-deltanrp1-and-wt" title="Permalink to this headline">¬∂</a></h1>
<div class="section" id="date">
<h2><span class="section-number">1.1. </span>Date<a class="headerlink" href="#date" title="Permalink to this headline">¬∂</a></h2>
<p>10082020 - 18082020</p>
</div>
<div class="section" id="objective">
<h2><span class="section-number">1.2. </span>Objective<a class="headerlink" href="#objective" title="Permalink to this headline">¬∂</a></h2>
<ul class="simple">
<li><p>To test the technique on the mutants and also to have data for the data analysis pipeline from Greg.</p></li>
</ul>
</div>
<div class="section" id="method">
<h2><span class="section-number">1.3. </span>Method<a class="headerlink" href="#method" title="Permalink to this headline">¬∂</a></h2>
<ul class="simple">
<li><p>10082020-Preculture in SD-URA+0.2%glucose +2% raffinose at 14:20</p>
<ul>
<li><p>Scrape from the colonies from each background that passed the test (2 per each background)</p></li>
<li><p>Inoculate to a final volume of 20mL and to a final OD of 0.2</p></li>
</ul>
</li>
<li><p><strong>11082020</strong></p>
<ul>
<li><p>OD measurement</p>
<ul>
<li><p>OD really low around 0.3 for all strains which could be due to the fact that the media did not contain extra adenine.</p></li>
<li><p>Add 6x ADE to the media which means: 44mg in 400mL .</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">SO</span> <span class="pre">I</span> <span class="pre">RESTARTED</span> <span class="pre">THE</span> <span class="pre">PRECULTURE</span> <span class="pre">ON</span> <span class="pre">11082020</span></code></p></li>
</ul>
</li>
</ul>
</li>
<li><p><strong>12082020</strong> - Induction in SD-URA+2% gal (I also added 6XAde to this media) to a final OD of 0.2 - 50 hours (2 days)</p>
<ul>
<li><p>OD measurement:</p></li>
</ul>
</li>
</ul>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>Strain</p></th>
<th class="head"><p>OD 10x dilution</p></th>
<th class="head"><p>ReaL OD</p></th>
<th class="head"><p>Dilution Factor to OD=0.2</p></th>
<th class="head"><p>Time of preculture</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>ylic133_1</p></td>
<td><p>0.659</p></td>
<td><p>6.6</p></td>
<td><p>~30</p></td>
<td><p>20h</p></td>
</tr>
<tr class="row-odd"><td><p>ylic133_2</p></td>
<td><p>0.577</p></td>
<td><p>5.7</p></td>
<td><p>~30</p></td>
<td><p>20h</p></td>
</tr>
<tr class="row-even"><td><p>ylic135_1</p></td>
<td><p>0.073</p></td>
<td><p>0.7</p></td>
<td><p>~3</p></td>
<td><p>20h</p></td>
</tr>
<tr class="row-odd"><td><p>ylic135_2</p></td>
<td><p>0.126</p></td>
<td><p>1.26</p></td>
<td><p>~6</p></td>
<td><p>20h</p></td>
</tr>
<tr class="row-even"><td><p>ylic136_1</p></td>
<td><p>0.678</p></td>
<td><p>6.7</p></td>
<td><p>~30</p></td>
<td><p>20h</p></td>
</tr>
<tr class="row-odd"><td><p>ylic136_2</p></td>
<td><p>0.626</p></td>
<td><p>6.3</p></td>
<td><p>~30</p></td>
<td><p>20h</p></td>
</tr>
</tbody>
</table>
<ul class="simple">
<li><p>Issues :</p>
<ul>
<li><p>The ylic135 strains grows a bit more slower than the rest which is expected due to the phenotype of not having bem2. But the  amount of culture I need to add in order to have a final OD of 0.2 in 150mL is 50 mL and I only had 20mL. Therefore the initial OD of the induction will be around 0.1 for this strain.</p></li>
</ul>
</li>
<li><p>Measure the background (T=0 of induction)</p>
<ul>
<li><p>[x] Plate 200ul of the inoculum in SD-ADE+ 2% dextrose (expect 40-80 colonies per 200uL)</p></li>
<li><p>[x] Dilute 1000x and spread 200ul in SD-URA</p></li>
</ul>
</li>
<li><p>[x] Prepare 3 flasks of 3L of SD-ADE media for the reseed of three strains</p>
<ul>
<li><p>Per flask of 3L , add: 2,4L MiliQ to autoclave, dissolve 2.34g of -ade drop out + 20.7g of YNB (005) and filter sterilize it and add it after autoclaving the MiliQ.  Add 300mL of 20% dextrose after autoclaving.</p></li>
</ul>
</li>
<li><p><strong>13082020</strong></p></li>
<li><p>[x] Plating in SD-ADE to check induction after T=22h and SD-URA 1000x dilution.</p></li>
<li><p>[x] Check OD (should be around 4-5) at 10:45am</p></li>
</ul>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>Strain</p></th>
<th class="head"><p>OD 10x dilution</p></th>
<th class="head"><p>ReaL OD</p></th>
<th class="head"><p>Time</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>ylic133_1</p></td>
<td><p>0.472</p></td>
<td><p>4.7</p></td>
<td><p>24h</p></td>
</tr>
<tr class="row-odd"><td><p>ylic133_2</p></td>
<td><p>1.17</p></td>
<td><p><strong>11.7</strong></p></td>
<td><p>24h</p></td>
</tr>
<tr class="row-even"><td><p>ylic135_1</p></td>
<td><p>0.223</p></td>
<td><p>2.23</p></td>
<td><p>24h</p></td>
</tr>
<tr class="row-odd"><td><p>ylic135_2</p></td>
<td><p>0.217</p></td>
<td><p>2.17</p></td>
<td><p>24h</p></td>
</tr>
<tr class="row-even"><td><p>ylic136_1</p></td>
<td><p>1.17</p></td>
<td><p><strong>11.7</strong></p></td>
<td><p>24h</p></td>
</tr>
<tr class="row-odd"><td><p>ylic136_2</p></td>
<td><p>1.13</p></td>
<td><p><strong>11.3</strong></p></td>
<td><p>24h</p></td>
</tr>
</tbody>
</table>
<ul class="simple">
<li><p>Strains ylic133_2 and ylic136 have a very high OD</p></li>
<li><p>The cell culture of ylic133_1 have turned pink‚Ä¶</p></li>
<li><p>Still the background at T=0 can not be clearly determined.</p></li>
<li><p><strong>14082020</strong> - End of induction/Reseed at 14:00</p>
<ul>
<li><p>Choose the 3 strains to continue for the reseed that has the least number of ADE+ cells at T=0 and the highest number of ADE+ after induction.</p>
<ul>
<li><p>It seems that ylic133_2, ylic136_1 and ylic136_2 are the ones that have the least background at T=0 before induction.</p></li>
</ul>
</li>
</ul>
</li>
<li><p>Background check at 9:30am</p></li>
</ul>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>Strain</p></th>
<th class="head"><p>ade+ clones/mL</p></th>
<th class="head"><p>ura+ clones/mL</p></th>
<th class="head"><p>Time</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>ylic133_1</p></td>
<td><p>&lt; 5</p></td>
<td><p>~100* 5* E3 = 5*E5</p></td>
<td><p>0h Induction</p></td>
</tr>
<tr class="row-odd"><td><p><strong>ylic133_2</strong></p></td>
<td><p>~30*5 = 150</p></td>
<td><p>5 *E5</p></td>
<td><p>0h Induction</p></td>
</tr>
<tr class="row-even"><td><p>ylic135_1</p></td>
<td><p>-</p></td>
<td><p>-</p></td>
<td><p>0h Induction</p></td>
</tr>
<tr class="row-odd"><td><p>ylic135_2</p></td>
<td><p>-</p></td>
<td><p>-</p></td>
<td><p>0h Induction</p></td>
</tr>
<tr class="row-even"><td><p><strong>ylic136_1</strong></p></td>
<td><p>~ 80*5=400</p></td>
<td><p>5 *E5</p></td>
<td><p>0h Induction</p></td>
</tr>
<tr class="row-odd"><td><p><strong>ylic136_2</strong></p></td>
<td><p>~ 80*5=400</p></td>
<td><p>5 *E5</p></td>
<td><p>0h Induction</p></td>
</tr>
</tbody>
</table>
<ul class="simple">
<li><p>Summary of the background</p></li>
</ul>
<p><img alt="" src="../../_images/14082020_-ade_background_all_strains.png" /></p>
<p><img alt="" src="../../_images/14082020_-ura_background_all_strains.png" /></p>
<ul class="simple">
<li><p>[x] Plate 200ul from the induction culture in -ade and 1000X in -ura to know the number of ade+ cells before reseeding. At T=51h of induction, at 13:00.</p></li>
<li><p>[x] OD measurement of the samples at T=51h of induction  and T<span class="math notranslate nohighlight">\(_r\)</span>=0</p></li>
<li><p>Summary from the Induction</p></li>
</ul>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>Strain</p></th>
<th class="head"><p>OD start T=0</p></th>
<th class="head"><p>OD T=24h</p></th>
<th class="head"><p>OD stop T=51h</p></th>
<th class="head"><p>Time-Induction</p></th>
<th class="head"><p>ADE+/ml-start</p></th>
<th class="head"><p>ADE+/ml-24H</p></th>
<th class="head"><p>ADE+/ml-stop</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>ylic133_1</p></td>
<td><p>0.2</p></td>
<td><p>4.7</p></td>
<td><p>5.4</p></td>
<td><p>51h</p></td>
<td><p>330</p></td>
<td><p>500</p></td>
<td><p>7500</p></td>
</tr>
<tr class="row-odd"><td><p>ylic133_2</p></td>
<td><p>0.2</p></td>
<td><p>11.7</p></td>
<td><p><strong>13.9</strong></p></td>
<td><p>51h</p></td>
<td><p>180</p></td>
<td><p>3500</p></td>
<td><p>5000</p></td>
</tr>
<tr class="row-even"><td><p>ylic135_1</p></td>
<td><p>0.1</p></td>
<td><p>2.23</p></td>
<td><p>6.5</p></td>
<td><p>51h</p></td>
<td><p>150</p></td>
<td><p>200</p></td>
<td><p>350</p></td>
</tr>
<tr class="row-odd"><td><p>ylic135_2</p></td>
<td><p>0.1</p></td>
<td><p>2.17</p></td>
<td><p>6.3</p></td>
<td><p>51h</p></td>
<td><p>50</p></td>
<td><p>75</p></td>
<td><p>100</p></td>
</tr>
<tr class="row-even"><td><p>ylic136_1</p></td>
<td><p>0.2</p></td>
<td><p>11.7</p></td>
<td><p><strong>13.8</strong></p></td>
<td><p>51h</p></td>
<td><p>600</p></td>
<td><p>5000</p></td>
<td><p>10000</p></td>
</tr>
<tr class="row-odd"><td><p>ylic136_2</p></td>
<td><p>0.2</p></td>
<td><p>11.3</p></td>
<td><p><strong>12.9</strong></p></td>
<td><p>51h</p></td>
<td><p>500</p></td>
<td><p>3000</p></td>
<td><p>7500</p></td>
</tr>
</tbody>
</table>
<p><img alt="" src="../../_images/17082020-induction_approx_stats.png" /></p>
<p><img alt="" src="../../_images/17082020-induction-plating-strains-for-reseeding.png" /></p>
<ul>
<li><p>[x] Reseed at 13:30</p></li>
<li><p><strong>17082020</strong></p></li>
<li><p>[x] Check ADE+ cells during induction. Counting colonies from each plate.</p>
<ul class="simple">
<li><p>Image J protocol:</p>
<ol class="simple">
<li><p>select image of interest: circle tool and edit-&gt; clear outside</p></li>
<li><p>Image -&gt;  Type -&gt; 16bits</p></li>
<li><p>Make sure to capture the maximum number of right colonies: Image -&gt; Adjust -&gt; Threshold . Make sure the colonies are all red.</p></li>
<li><p>Process -&gt; Make binary-&gt; Watershed</p></li>
<li><p>Analyse -&gt;  Analyse Particles</p></li>
</ol>
</li>
</ul>
</li>
<li><p><strong>18082020</strong></p></li>
<li><p>[x] OD measurement for T=92h after reseeding at 12:00</p></li>
<li><p>[x] Plate 200ul with 1000X dilution in SD-Ade , and 200ul with 40000X dilution in SD-ura to estimate the growth of ade+cells compared with the T=0 of reseeding. Expect  that the ade+ cells have grown by a factor of ~1000x.</p></li>
<li><p>Reseeding</p>
<ul class="simple">
<li><p>Library to reseed</p></li>
<li><p><a class="reference external" href="https://research.fhcrc.org/content/dam/stripe/hahn/methods/yeast_genetics/yeast_OD_cells.pdf">Rule of thumb for the number of cells under certain OD600</a></p></li>
</ul>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>OD</p></th>
<th class="head"><p>approx Number of cells in 1mL</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>0.1</p></td>
<td><p>3,000,000</p></td>
</tr>
<tr class="row-odd"><td><p>0.18</p></td>
<td><p>6,000,000</p></td>
</tr>
<tr class="row-even"><td><p>0.25</p></td>
<td><p>7,500,000</p></td>
</tr>
<tr class="row-odd"><td><p>0.27</p></td>
<td><p>7,500,000</p></td>
</tr>
<tr class="row-even"><td><p>1</p></td>
<td><p>30,000,000</p></td>
</tr>
</tbody>
</table>
</li>
</ul>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>Strain</p></th>
<th class="head"><p>OD at T=0 (reseed)</p></th>
<th class="head"><p>Volume</p></th>
<th class="head"><p>Library</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>ylic133_2</p></td>
<td><p>0.25</p></td>
<td><p>3L</p></td>
<td><p>7,500,000 * 3 * <span class="math notranslate nohighlight">\(10^3\)</span>= <span class="math notranslate nohighlight">\(22.5 *10^9\)</span></p></td>
</tr>
<tr class="row-odd"><td><p>ylic136_1</p></td>
<td><p>0.27</p></td>
<td><p>3L</p></td>
<td><p>7,500,000 * 3 * <span class="math notranslate nohighlight">\(10^3\)</span>= <span class="math notranslate nohighlight">\(22.5 *10^9\)</span></p></td>
</tr>
<tr class="row-even"><td><p>ylic136_2</p></td>
<td><p>0.18</p></td>
<td><p>3L</p></td>
<td><p>6,000,000* 3 * <span class="math notranslate nohighlight">\(10^3\)</span>= <span class="math notranslate nohighlight">\(18 *10^9\)</span></p></td>
</tr>
</tbody>
</table>
<ul class="simple">
<li><p>End of reseeding</p></li>
<li><p>[x] Harvest of the cell culture.</p>
<ul>
<li><p>15ml of solid pellet, frozen in -80C. The pellet is pink (?)
<img alt="" src="../../_images/19082020-pink-cell-pellet.png" /></p></li>
</ul>
</li>
</ul>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>Strain</p></th>
<th class="head"><p>OD START</p></th>
<th class="head"><p>OD STOP</p></th>
<th class="head"><p>ADE+/mL-start<br>induct(backg)</p></th>
<th class="head"><p>ADE+/mL-start<br>Reseed</p></th>
<th class="head"><p>ADE+/ml-stop<br>Reseed</p></th>
<th class="head"><p>URA+/ml-stop<br>Reseed</p></th>
<th class="head"><p>% of ade+/ura+</p></th>
<th class="head"><p>Total # of cells<br>Harvest</p></th>
<th class="head"><p>Time- Reseeding</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>ylic133_2</p></td>
<td><p>0.25</p></td>
<td><p>8.3</p></td>
<td><p>150</p></td>
<td><p>5000</p></td>
<td><p>3295000</p></td>
<td><p>19200000</p></td>
<td><p>16.9</p></td>
<td><p><span class="math notranslate nohighlight">\(7.47 *10^{11}\)</span></p></td>
<td><p>90h</p></td>
</tr>
<tr class="row-odd"><td><p>ylic136_1</p></td>
<td><p>0.27</p></td>
<td><p>10.2</p></td>
<td><p>400</p></td>
<td><p>10000</p></td>
<td><p>5350000</p></td>
<td><p>18000000</p></td>
<td><p>27.8</p></td>
<td><p><span class="math notranslate nohighlight">\(9.18*10^{11}\)</span></p></td>
<td><p>90h</p></td>
</tr>
<tr class="row-even"><td><p>ylic136_2</p></td>
<td><p>0.18</p></td>
<td><p>9.5</p></td>
<td><p>400</p></td>
<td><p>7500</p></td>
<td><p>3070000</p></td>
<td><p>11600000</p></td>
<td><p>30.2</p></td>
<td><p><span class="math notranslate nohighlight">\(8.55*10^{11}\)</span></p></td>
<td><p>90h</p></td>
</tr>
</tbody>
</table>
<p><img alt="" src="../../_images/20082020-stats-reseeding.png" /></p>
<div class="section" id="defining-the-complexity-of-the-library">
<h3><span class="section-number">1.3.1. </span>Defining the complexity of the library<a class="headerlink" href="#defining-the-complexity-of-the-library" title="Permalink to this headline">¬∂</a></h3>
<ul class="simple">
<li><p>The number of ADE+ cells reseeded minus the number of background ADE+ cells is the complexity of the library.</p></li>
<li><p>To determine the complexity of the library, you should know:</p>
<ul>
<li><p>#ADE+ per ml at the start of the reseed,</p></li>
<li><p>Volume of reseed.</p></li>
<li><p>If you harvest all the culture:</p></li>
</ul>
</li>
</ul>
<p>For strain 1:</p>
<p>OD8 ~8E+7 cells/ml -&gt; 7.47E+11/ 8E+7~10L</p>
<ul>
<li><p>The  number of URA+ cells per ml at the end of the regrowth is 19200000 -&gt; 7.47E+11/ 19.2E+5 ~ 38L</p></li>
<li><p>I reseed in a volume of 3L :</p>
<p>If so, the complexity of the library is 5E+3 * 3*10E+3= 1.5E+7</p>
</li>
<li><p>Complexity = <span class="math notranslate nohighlight">\(ADE+|_{T=0(reseed)}\)</span> * Volume of reseed</p></li>
</ul>
</div>
<div class="section" id="number-of-cycles-during-the-reseed">
<h3><span class="section-number">1.3.2. </span>Number of cycles during the reseed<a class="headerlink" href="#number-of-cycles-during-the-reseed" title="Permalink to this headline">¬∂</a></h3>
<ul class="simple">
<li><p>The number of cycles that each ADE+ cells has done during the regrowth of the libraries as follow:</p>
<ul>
<li><p>N=ln(#ADE+ per ml STOP reseed/#ADE+ per ml START reseed)/ln(2)</p></li>
<li><p>N_ WT=9.34</p></li>
<li><p>N_dnrp1_1=9.06</p></li>
<li><p>N_dnrp1_2=8.67</p></li>
</ul>
</li>
</ul>
</div>
</div>
<div class="section" id="results">
<h2><span class="section-number">1.4. </span>Results<a class="headerlink" href="#results" title="Permalink to this headline">¬∂</a></h2>
<ul class="simple">
<li><p>For the strain ylic135 it did not work, because it is very miserable to culture with others. It grows much more slower , so maybe growing it more and with less volume , for next time. Also ylic135 gives a relatively high background compared with the growth in -URA.</p></li>
<li><p>The way of computing the number of colonies in highly dense plates is pretty inaccurate with the best tool to do it which is ImageJ. So I think that is why I get like an order of magnitude lower than the number of colonies expected by Benoit.</p></li>
<li><p>The pellet from the reseed culture was pink , which could mean that there were many cells ade- that did not have any transposon event. Hence I suspect the sequencing/transposon coverage will be not optimal.</p></li>
<li><p>The number of ADE+ cells after reseeding is around 1000X times higher than at then of the induction which is what was expected :)</p></li>
</ul>
</div>
<div class="section" id="next-steps">
<h2><span class="section-number">1.5. </span>Next steps<a class="headerlink" href="#next-steps" title="Permalink to this headline">¬∂</a></h2>
<ul class="simple">
<li><p>DNA sequencing protocol</p></li>
<li><p>Repeat the SATAY for ylic135 from the beginning</p></li>
</ul>
</div>
<div class="section" id="conclusion">
<h2><span class="section-number">1.6. </span>Conclusion<a class="headerlink" href="#conclusion" title="Permalink to this headline">¬∂</a></h2>
<ul class="simple">
<li><p>First round of SATAY :) with WT and dnrp1 strains.</p></li>
<li><p>Next time I will:</p>
<ul>
<li><p>Also plate the cells in CSM in order to capture the whole population.</p></li>
<li><p>Also plate the cells during the induction in -URA mainly to know the number of cells before the reseeding and to quantify better the ADe+/ADE- ratio across induction.</p></li>
</ul>
</li>
</ul>
</div>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./journal\2020-08"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
        </div>
    </div>
    
    
    <div class='prev-next-bottom'>
        
    <a class='left-prev' id="prev-link" href="../2020-10/2020-10-21-Digestion-Ligation2Sequence.html" title="previous page"><span class="section-number">4. </span>Title : Digestion and Ligation for sequencing</a>
    <a class='right-next' id="next-link" href="2020-08-18-DNA-Prep-SATAY.html" title="next page"><span class="section-number">2. </span>DNA extraction from SATAY samples</a>

    </div>
    <footer class="footer mt-5 mt-md-0">
    <div class="container">
      <p>
        
          By Leila I√±igo de la Cruz<br/>
        
            &copy; Copyright Creative Commons.<br/>
      </p>
    </div>
  </footer>
</main>


      </div>
    </div>

    
  <script src="../../_static/js/index.3da636dd464baa7582d2.js"></script>


    
  </body>
</html>